---

title: Modified forms of exotoxin A
abstract: exotoxin A or “PE” is a 66kD, highly potent, cytotoxic protein secreted by the bacterium . Various forms of PE have been coupled to other proteins, such as antibodies, to generate therapeutically useful cytotoxin conjugates that selectively target cells of a desired phenotype (such as tumor cells). In the present invention, peptides spanning the sequence of an approximately 38kD form of exotoxin A protein were analyzed for the presence of immunogenic CD4+ T cell epitopes. Six immunogenic T cell epitopes were identified. Residues were identified within each epitope for introduction of targeted amino acid substitutions to reduce or prevent immunogenic T-cell responses in PE molecules which may be administered to a heterologous host.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09447387&OS=09447387&RS=09447387
owner: INTREXON CORPORATION
number: 09447387
owner_city: Blacksburg
owner_country: US
publication_date: 20151230
---
This application is a continuation of U.S. application Ser. No. 14 561 707 filed Dec. 5 2014 which is a continuation of U.S. application Ser. No. 13 604 173 filed Sep. 5 2012 now U.S. Pat. No. 8 932 586 which claims priority benefit of U.S. Application No. 61 531 576 filed Sep. 6 2011.

This application claims benefit of and priority based on U.S. Provisional Patent Application Ser. No. 61 531 576 filed Sep. 6 2011 the contents of which are herein incorporated by reference in their entirety.

The claimed invention was made pursuant to a joint research agreement as defined in 35 U.S.C. 103 c 3 that was in effect on or before the date the claimed invention was made and as a result of activities undertaken within the scope of the joint research agreement by or on behalf of the Intrexon Corp. Foster City Calif. U.S.A. and Antitope Ltd. Cambridge UK .

A Sequence Listing is submitted electronically via EFS Web as an ASCII formatted sequence listing in a file named OT050 PCT SEQLIST.txt created on Sep. 4 2012 and having a file size of 295 678 bytes which is filed concurrently with the present specification claims abstract and figures provided herewith. The sequence listing contained in this ASCII formatted document is part of the specification and is herein incorporated by reference in its entirety.

Immune responses to biological therapeutic agents are wide ranging and can be directed against agents that are both non human and human in origin. These responses include those that elicit a weak clinical effect and those that limit efficacy which can occasionally result in morbidity or even mortality in patients. In particular serious complications can arise with the production of neutralizing antibodies especially when they target recombinant self proteins and therefore have the potential to cross react with the patient s own endogenous protein Lim 2005 . Problems associated with immunogenicity to biologics i.e. therapeutic medical products such as antibodies and recombinant proteins polypeptides have been reduced largely due to advances in molecular biology. There are however many recombinant protein biologics that are identical to endogenously expressed human sequences that still elicit potent neutralizing immune responses in patients Hochuli 1997 Schellekens et al 1997 Namaka et al 2006 . The mechanism by which immunogenicity is triggered remains unclear although the tolerance to self proteins may be broken by a number of factors linked to both the product and the patient reviewed in Chester et a 2006 Baker and Jones 2007 . For the product these include dose frequency of administration route immunomodulatory capacity of the protein therapeutic and the formulation Jaber and Baker 2007 . For the patient factors such as immune competence i.e. whether the patient is receiving immunosuppressive treatment patient s MHC haplotype and intrinsic tolerance to the protein therapeutic will influence immunogenicity. Regardless of how immunogenicity is triggered one of the single most important factors in the development of an ensuing immune response is the presence of epitopes that are able to effectively stimulate a potent CD4 T cell response reviewed Baker and Jones 2007 .

T cells or T lymphocytes are a subset of white blood cells known as lymphocytes. The abbreviation T in T cell is for thymus since this is the primary organ responsible for T cell maturation. T cells play a central role in cell mediated immunity. They can be distinguished from other types of lymphocytes such as B cells and natural killer cells NK cells by the presence of cell surface proteins called T cell receptors TCRs . Different types of T cells have also been identified these can be distinguished based on the differing functions they serve e.g. CD4 T cells a.k.a. Tor T helper cells CD8 cytotoxic T cells CTLs memory T cells regulatory T cells Tcells natural killer cells NK cells and gamma delta T cells T cells .

T helper T cells are so named because they aid other white blood cells in immunologic processes including inter alia assisting the maturation of B cells into plasma and B memory cells and activation of cytotoxic T cells and macrophages. Tcells are also known as CD4 T cells because they express CD4 protein on the cell surface. CD4 T cells are activated when peptide antigens are presented by MHC class II molecules expressed on the surface of Antigen Presenting Cells APCs . Once activated CD4 T cells divide rapidly and secrete chemokines that further assist in activating or regulating immune responses.

T cell epitope analysis is becoming increasingly important particularly in the pre clinical analysis of biologics and may in time become a requirement for regulatory approval for clinical trials. To this end a pre clinical ex vivo T cell assay EPISCREEN has been used to provide an effective technology for predicting T cell immunogenicity by identifying linear T cell epitopes present in protein sequences. Synthetic overlapping peptides typically of about 15 amino acids in length are tested against a cohort of community blood donors carefully selected based on MHC class II haplotypes to provide a quantitative analysis of T cell epitopes present in protein sequences. This technology has been used successfully to compare protein variants for the potential to induce an immune response in vivo. By providing a high degree of sensitivity along with high reproducibility the EPISCREEN assay allows an accurate pre clinical assessment of the potential for immunogenicity of biologics. See Baker Carr Preclinical Considerations in the Assessment of Immunogenicity for Protein Therapeutics 5 4 1 6 2010 Bryson et al. Prediction of Immunogenicity of Therapeutic Proteins Validity of Computational Tools 24 1 1 8 2010 Holgate Baker Circumventing Immunogenicity in the Development of Therapeutic Antibodies 12 4 233 237 2009 Perry et al. New Approaches to Prediction of Immune Responses to Therapeutic Proteins during Preclinical Development 9 6 385 396 2008 and Baker Jones Identification and removal of immunogenicity in therapeutic proteins 10 2 219 227 2007 .

PE A has at least three different structural domains responsible for various biological activities . See e.g. Siegall et al. vol. 30 pp. 7154 7159 1991 Theuer et al. . vol. 267 no. 24 pp. 16872 16877 1992 and U.S. Pat. No. 5 821 238. PE A domain IA amino acids 1 252 see e.g. SEQ ID NO 133 is responsible for cell binding. Domain II amino acids 253 364 see e.g. SEQ ID NO 133 is responsible for translocation of PE A into the cell cytosol. Domain III the cytotoxic domain amino acids 400 613 see e.g. SEQ ID NO 133 is responsible for ADP ribosylation of Elongation Factor 2 EF2 which thereby inactivates EF2 subsequently causing cell death. Additionally a function for domain IB amino acids 365 399 SEQ ID NO 139 has not been established. Indeed it has been reported that amino acids 365 380 SEQ ID NO 138 within domain IB can be deleted without producing an identifiable a loss of function. See Siegall et al. vol. 30 pp. 7154 7159 1991 .

It has also been reported that PE A may comprise any one of at least three different carboxy terminal tails these appear to be essential for maintaining or recycling proteins into the endoplasmic reticulum. See Theuer et al. . vol. 267 no. 24 pp. 16872 16877 1992 Chaudhary et al. vol. 87 pp. 308 312 1990 and Seetharam et al. . vol. 266 17376 17381 1991 . In particular in correspondence with the exemplary sequence shown in SEQ ID NO 133 these alternative carboxy terminal tails comprise amino acid sequences 

Variants of PE A modified to lack the cell binding domain but coupled to heterologous cell specific targeting molecules e.g. antibodies have been shown to have reduced levels of non specific toxicity. See e.g. U.S. Pat. No. 4 892 827.

Various forms of PE A e.g. truncated deletion forms with molecular weights of 37 kD 38 kD 40 kD et cetera have been combined with a number of growth factors antibodies and other proteins to generate cytotoxins which selectively target cells of a desired phenotype. See for example 

A significant disadvantage in using PE A for treatment of disease however is that it is a foreign non self protein being introduced into a heterologous host e.g. a human .

Introduction of non self proteins into heterologous hosts commonly elicits host immune reactions such as the generation of antibodies neutralizing antibodies or immune cell reactions e.g. cytotoxic T cell responses which are directed at eliminating the non self protein i.e. PE A . Accordingly it would be advantageous if elements of PE A PE A epitopes which are recognized and targeted as non self could be removed prior to use of this molecule as a therapeutic agent.

Some investigators have previously attempted to identify and remove immunogenic determinants from PE A i.e. to deimmunize PE A . See for example 

Despite progress in the area of deimmunization of PE A there remains a need for the development of optimized less immunogenic or non immunogenic biologically active forms of this useful cytotoxin. The invention described herein addresses this need.

Peptides spanning the sequence of an approximately 38 kD predicted molecular weight form of exotoxin A protein SEQ ID NO 1 were analyzed for the presence of immunogenic CD4 T cell epitopes. A total of 120 overlapping 15mer peptides spanning this sequence SEQ ID NO 1 but also including an amino terminal Gly Ser linker sequence SEQ ID NO 3 to produce a 359 amino acid Gly Ser PE38 polypeptide sequence SEQ ID NO 2 were tested against a cohort of healthy human donors. CD4 T cell responses against individual peptides were measured via proliferation assays. Assay data was used to compile a T cell epitope map of the PE38 sequence. Six immunogenic T cell epitopes were identified. Residues were then identified within each of these epitopes for use in targeted amino acid substitutions to reduce or prevent PE38 induced immunogenicity. Reduction or prevention of PE immunogenicity should allow for multiple therapeutic administrations of cytotoxic PE for use for example in the targeted destruction of cancer cells in vivo such as when administered as an immunoconjugate or cell surface targeted fusion protein .

Unless specifically indicated otherwise as used herein the term PE or PE A is intended to indicate a polypeptide comprising a cytotoxic polypeptide sequence derived from a wild type or naturally occurring form of exotoxin A protein. In addition to cytotoxic polypeptide sequences PE polypeptides may comprise additional naturally occurring or heterologous polypeptide sequences. Additional naturally occurring polypeptide sequences may include sequences such as are found in full length exotoxin A protein for example amino acid sequences responsible for cytosolic translocation and cell specific targeting as discussed further herein . Additional heterologous polypeptide sequences may include sequences with which at least a PE cytotoxic polypeptide is fused to impart additional functions or properties. For example a PE cytotoxic polypeptide may be fused to antigen binding polypeptide sequences such as an scFv antibody. Examples of sequences comprising a cytotoxic portion of PE can be found in SEQ ID NO 1 and SEQ ID NO 4 spanning amino acid residues Phe 134 to Lys 347. Examples of sequences comprising a cytotoxic portion of PE can also be found in SEQ ID NO 133 and SEQ ID NO 134 spanning amino acid residues Phe 400 to Lys 613.

As used herein in reference to PE unless indicated otherwise a cytotoxic polypeptide or cytotoxic polypeptide sequence is intended to indicate a polypeptide or portion thereof which is capable of inactivating translation elongation factor 2 EF 2 mediating ADP ribosylation of EF 2 inhibiting protein synthesis or inducing cell death. For example it has been demonstrated that PE domain III comprised of amino acid residues 400 613 of SEQ ID NO 133 is sufficient to mediate ADP ribosylation of EF 2 and thereby cause cell death. See Theuer et al. . vol. 267 no. 24 pp. 16872 16877 1992 and Hwang et al. 48 pp. 129 136 1987 .

Cytotoxic polypeptide sequences in the present invention may also comprise alternative carboxy terminal sequences. See Theuer et al. Chaudhary et al. and Seetharam et al. In particular embodiments examples of carboxy terminal tails of PE38 in the present invention may comprise sequences as shown in SEQ ID NO 133 . Hence exemplary alternative carboxy terminal tails may comprise amino acid sequences 

Unless specifically indicated otherwise as used herein the term PE38 is intended to indicate a exotoxin A PE or PE A molecule comprising an amino acid sequence as shown in SEQ ID NO 1. The amino acid sequence used to generate peptide sequences referenced in the Examples is shown in SEQ ID NO 2. SEQ ID NO 2 comprises an amino terminal GGGGGSGGGGGS linker sequence SEQ ID NO 3 fused to the PE38 amino acid sequence of SEQ ID NO 1. A variant form of PE38 is shown in SEQ ID NO 4. SEQ ID NO 4 differs from SEQ ID NO 1 by comprising a Ser to Asn change at position 114 a Ile to Val change at position 141 and a Gly to Ser change at position 249.

As used herein unless specifically stated otherwise biological activity in reference to exotoxin A PE A PE or PE38 is intended to indicate at least one of the biological activities exhibited by naturally occurring forms of the exotoxin A molecule. These activities include for example cell killing or cell cytotoxic activity a.k.a. cell cytotoxicity inactivation of translation elongation factor EF 2 ADP ribosylation of EF 2 and inhibition of protein synthesis. The biological activity of PE and PE38 polypeptides and modified forms thereof e.g. PE and PE38 amino acid substituted variants and fusion proteins can be measured using assays and experiments which are well known and routinely used by those skilled in the art. Examples of some of these assays and experiments are further described and referenced herein without limitation in the Examples sections included herein.

As used herein the term having exotoxin A PE A biological activity or PE biological activity is intended to indicate molecules exhibiting about 5 or more of at least one biological activity compared to a corresponding wild type naturally occurring or non amino acid substituted form of PE or PE A. In some embodiments molecules having exotoxin A biological activity or PE biological activity exhibit 5 or more about 10 or more 10 or more about 15 or more 15 or more about 20 or more 20 or more about 25 or more 25 or more about 30 or more 30 or more about 35 or more 35 or more about 40 or more 40 or more about 45 or more 45 or more about 50 or more 50 or more about 60 or more 60 or more about 70 or more 70 or more about 75 or more 75 or more about 80 or more 80 or more about 85 or more 85 or more about 90 or more 90 or more about 95 or more 95 or more about 100 or 100 of at least one biological activity compared to a corresponding wild type naturally occurring or non amino acid substituted forms of PE or PE A.

As used herein the term wild type exotoxin A PE A or wild type PE biological activity is intended to indicate at least one or more biological activities exhibited by naturally occurring forms of the exotoxin A PE A or PE polypeptides. These include for example without limitation activities such as cell killing or cell cytotoxic activity a.k.a. cell cytotoxicity inactivation of translation elongation factor EF 2 ADP ribosylation of EF 2 and inhibition of protein synthesis. Two examples without limitation of polypeptide sequences representing wild type or non amino acid substituted forms of PE A are shown in SEQ ID NO 133 and SEQ ID NO 134. Two examples without limitation of polypeptide sequences representing wild type or non amino acid substituted forms of PE are shown in SEQ ID NO 1 PE38 and SEQ ID NO 4 variant of PE38 .

As used or claimed herein the term a or an in reference to the subsequent recited entity refers to one or more of that entity for example a PE38 antibody or a polynucleotide encoding PE38 is understood to indicate one or more PE38 antibody molecules and one or more polynucleotides encoding PE38 not a single PE38 antibody molecule nor a single polynucleotide molecule encoding PE38 respectively. As such the terms a or an one or more and at least one can be used interchangeably herein.

Likewise as used herein the term polypeptide is intended to encompass a singular polypeptide as well as plural polypeptides and refers to a molecule composed of monomers amino acids linked by amide bonds also known as peptide bonds . The term polypeptide refers to any chain or chains of two or more amino acids and does not refer to a specific length of the product. Thus peptides dipeptides tripeptides oligopeptides protein amino acid chain or any other term used to refer to a chain or chains of two or more amino acids are included within the definition of polypeptide and the term polypeptide may be used instead of or interchangeably with any of these terms. The term polypeptide is also intended to refer to the products of post expression modifications of the polypeptide such as but without limitation glycosylation acetylation phosphorylation amidation et cetera. A polypeptide unless specifically described otherwise herein may be derived from a natural biological source or produced by recombinant technology but is not necessarily translated from a designated nucleic acid sequence. It may be generated in any manner including by chemical synthesis.

Polypeptides may have a defined three dimensional structure although they do not necessarily have such structure. Polypeptides with a defined three dimensional structure may be referred to as folded or having a tertiary structure. Polypeptides not configured into a three dimensional structure are referred to as unfolded. As used herein the term glycoprotein refers to a protein coupled to at least one carbohydrate moiety attached to the protein via a covalent bond.

The term isolated is intended to indicate a biological component no longer in its naturally occurring milieu. For example an isolated polypeptide or isolated polynucleotide is intended to indicate a polypeptide or polynucleotide respectively which has been removed from its naturally occurring milieu and which may have been inserted within a non naturally occurring milieu. By way of example this would include without limitation a polynucleotide which has been removed from a naturally occurring location within a host genome and subsequently inserted for example into an expression vector or inserted into a new host genome location or into the genome of a heterologous host organism. The isolation of a polypeptide or polynucleotide as used herein requires no particular level of purification. For example recombinantly produced polypeptides expressed in host cells are considered isolated for purposes of the invention as are native or recombinant polypeptides which have been separated fractionated or partially or substantially purified by any suitable technique.

Polypeptide embodiments also include fragments derivatives analogs variants and fusion proteins preferably but not necessarily wherein such embodiments retain one or more biological activities associated with a corresponding full length or naturally occurring polypeptide. Fragments include proteolytic fragments deletion fragments and fragments encoded by synthetically or recombinantly produced polynucleotides. Variants may occur naturally or be non naturally occurring. Non naturally occurring variants may be produced using art known mutagenesis techniques. Variant polypeptides may comprise conservative or non conservative amino acid substitutions deletions or additions. Derivatives include but are not limited to polypeptides which contain one or more non naturally occurring amino acids non standard amino acids and amino acid analogs. Polypeptide embodiments may comprise amino acid sequences which are at least 60 identical at least 70 identical at least 80 identical at least 85 identical at least 90 identical at least 95 identical at least 97 identical at least 98 identical or at least 99 identical to SEQ ID NO 1.

Unless specifically defined otherwise the term polynucleotide is intended to indicate nucleic acid molecules or constructs as routinely used and understood by those of skill in the art. For example nucleic acids include but are not limited to molecules such as messenger RNA mRNA plasmid DNA pDNA complementary DNA cDNA and genomic DNA gDNA . A polynucleotide may comprise a conventional phosphodiester bond or a non conventional bond e.g. an amide bond such as found in peptide nucleic acids PNA . The terms polynucleotide and nucleic acid are intended to include embodiments wherein any one or more sequences of polynucleotide or nucleic acid segments are contained or comprised within a larger polynucleotide or nucleic acid sequence. For example but without limitation and unless stated otherwise to the contrary herein reference to a nucleic acid such as a polynucleotide encoding PE38 is intended to include nucleic acids comprising a polynucleotide encoding PE38 wherein such polynucleotide may also be part of a larger nucleic acid or polynucleotide such as an expression vector or a polynucleotide nucleic acid encoding an PE fusion protein.

An isolated nucleic acid or polynucleotide is intended a nucleic acid molecule DNA or RNA which has been removed from its native environment. For example a recombinant polynucleotide encoding an antibody contained in a vector is considered isolated for the purposes of the present invention. Further examples of an isolated polynucleotide include recombinant polynucleotides maintained in heterologous host cells or purified partially or substantially polynucleotides in solution. Isolated RNA molecules include in vivo or in vitro synthesized RNA molecules including synthetically produced molecules.

As used herein a coding region is a portion of nucleic acid containing codons which may be translated into amino acids although stop codons TAG TGA or TAA are not translated into an amino acids but may also be considered to be part of a coding region. Unless stated otherwise herein promoters ribosome binding sites transcriptional terminators introns and the like are not considered part of a coding region. Two or more coding regions of the present invention can be present in a single polynucleotide construct e.g. on a single vector or in separate polynucleotide constructs e.g. on separate different vectors. Furthermore any vector may contain a single coding region or may comprise two or more coding regions e.g. a single vector may separately encode an immunoglobulin heavy chain variable region and an immunoglobulin light chain variable region. In addition a vector polynucleotide or nucleic acid embodiments may encode heterologous coding regions either fused or unfused to a nucleic acid encoding a different heterologous polypeptide. Heterologous coding regions include without limitation specialized elements or motifs such as a secretory signal peptide or a heterologous functional domains.

In certain embodiments the polynucleotide or nucleic acid is DNA. In the case of DNA a polynucleotide comprising a nucleic acid which encodes a polypeptide normally may include a promoter and or other transcription or translation control elements operably associated with one or more coding regions. An operable association is when a coding region for a gene product e.g. a polypeptide is associated with one or more regulatory sequences in such a way as to place expression of the gene product under the influence or control of the regulatory sequence s . Two DNA fragments such as a polypeptide coding region and a promoter associated therewith are operably associated if induction of promoter function results in the transcription of mRNA encoding the desired gene product. Thus a promoter region would be operably associated with a nucleic acid encoding a polypeptide if the promoter was capable of effecting transcription of that nucleic acid. Other transcription control elements besides a promoter include for example but without limitation enhancers operators repressors and transcription termination signals can be operably associated with the polynucleotide to direct cell specific transcription. Suitable promoters and other transcription control regions are disclosed herein.

The terms antibody and immunoglobulin may be used interchangeably herein. An antibody or immunoglobulin comprises at least the antigen binding elements e.g. complementarity determining regions or CDRs of the variable domain of a heavy chain and or of the variable domain of a light chain. Basic immunoglobulin structures in vertebrate systems are well understood by those of skill in the art. See e.g. Harlow Lane Using Antibodies A Laboratory Manual Cold Spring Harbor Laboratory Press 1999 ISBN 0879695447 see also Harlow Lane Antibodies A Laboratory Manual Cold Spring Harbor Laboratory Press 2nd ed. 1988 . The term immunoglobulin or antibody comprises various broad classes of antibody molecules such as but without limitation IgG IgM IgA IgG and IgE classes of antibodies as well as antibody subclasses isotypes such as IgG1 IgG2 IgG3 IgG4 IgA1 et cetera.

Antibodies or antigen binding fragments variants or derivatives thereof of the invention include but are not limited to polyclonal monoclonal multispecific human humanized primatized or chimeric antibodies single chain antibodies epitope binding fragments e.g. Fab Fab and F ab Fd Fvs single chain Fvs scFv single chain antibodies disulfide linked Fvs sdFv fragments comprising either a VL or VH domain fragments produced by a Fab expression library and anti idiotypic anti Id antibodies.

As used herein an epitope or antigenic determinant is the part of a polypeptide antigen or molecule that is recognized by the immune system specifically by antibodies B cells or T cells. Epitopes of polypeptide antigens may function as conformational epitopes or linear epitopes. A conformational epitope is comprised of non linear sections of a target molecule such as that formed via the tertiary structure of a folded polypeptide . In contrast amino acids that make up a linear epitope may be comprised of a continuous sequence of amino acids or may be comprised only of particular amino acid residues critical to antibody B cell T cell binding.

By specifically binds it is generally meant that an antibody binds to an epitope via its antigen binding domain and that the binding entails some complementarity between the antigen binding domain and the epitope. According to this definition an antibody is said to specifically bind to an epitope when it binds to that epitope via its antigen binding domain more readily than it would bind to a random unrelated epitope. The term specificity may be used herein to qualify the relative affinity by which a certain antibody binds to a certain epitope. For example antibody A may be deemed to have a higher specificity for a given epitope than antibody B or antibody A may be said to bind to epitope C with a higher specificity than it has for related epitope D. 

By preferentially binds it is meant that the antibody specifically binds to an epitope more readily than it would bind to a related similar homologous or analogous epitope. Thus an antibody which preferentially binds to a given epitope would more likely bind to that epitope than to a related epitope even though such an antibody may cross react with the related epitope.

An antibody is said to competitively inhibit binding of a reference antibody to a given epitope if it preferentially binds to that epitope to the extent that it blocks to some degree binding of the reference antibody to the epitope. Competitive inhibition may be determined by any method known in the art for example competition ELISA assays. An antibody may be said to competitively inhibit binding of the reference antibody to a given epitope by at least 90 at least 80 at least 70 at least 60 or at least 50 .

As used herein the term affinity refers to a measure of the strength of the binding of an individual epitope with the CDR of an immunoglobulin molecule. See e.g. Harlow et al. Antibodies A Laboratory Manual Cold Spring Harbor Laboratory Press 2nd ed. 1988 at pages 27 28. As used herein the term avidity refers to the overall stability of the complex between a population of immunoglobulins and an antigen that is the functional combining strength of an immunoglobulin mixture with the antigen. See e.g. Harlow at pages 29 34. Avidity is related to both the affinity of individual immunoglobulin molecules in the population with specific epitopes and also the valencies of the immunoglobulins and the antigen. For example the interaction between a bivalent monoclonal antibody and an antigen with a highly repeating epitope structure such as a polymer would be one of high avidity.

The term cross reactivity refers to the ability of an antibody specific for one antigen to react with a second antigen a measure of relatedness between two different antigenic substances. Thus an antibody is cross reactive if it binds to an epitope other than the one that induced its formation. The cross reactive epitope generally contains many of the same complementary structural features as the inducing epitope and in some cases may actually fit better than the original.

As used herein the terms linked fused or fusion may be used interchangeably. These terms refer to the joining together of two more elements or components by whatever means including chemical conjugation or recombinant means. An in frame fusion refers to the joining of two or more polynucleotide open reading frames ORFs to form a continuous longer ORF in a manner that maintains the correct translational reading frame of the original ORFs. Thus a recombinant fusion protein is a single protein containing two ore more segments that correspond to polypeptides encoded by the original ORFs which segments are not normally so joined in nature. Although the reading frame is thus made continuous throughout the fused segments the segments may be physically or spatially separated by for example in frame linker sequence. For example polynucleotides encoding the CDRs of an immunoglobulin variable region may be fused in frame but be separated by a polynucleotide encoding at least one immunoglobulin framework region or additional CDR regions as long as the fused CDRs are co translated as part of a continuous polypeptide.

In the context of polypeptides a linear sequence or a sequence is an order of amino acids in a polypeptide in an amino to carboxyl terminal direction in which residues that neighbor each other in the sequence are contiguous in the primary structure of the polypeptide.

A variant of a polypeptide or protein refers to any analogue fragment derivative or mutant which is derived from a polypeptide or protein and which retains at least one biological property of the polypeptide or protein. Different variants of the polypeptide or protein may exist in nature or may be generated artificially e.g. via synthetic or genetic engineering . Variants may be allelic variations characterized by differences in the nucleotide sequences of the structural gene coding for the protein or may involve differential splicing or post translational modification. The skilled artisan can produce variants having single or multiple amino acid substitutions deletions additions or replacements. Variants may include inter alia a variants in which one or more amino acid residues are substituted with for example conservative amino acids non conservative amino acids or amino acid analogs b variants in which one or more amino acids are added to the polypeptide or protein c variants in which one or more of the amino acids includes a substituent group and d variants in which the polypeptide or protein is fused with another polypeptide such as serum albumin. The techniques for obtaining these variants including genetic suppressions deletions mutations etc. chemical and enzymatic techniques are known to those of skill in the art.

The term expression as used herein refers to a process by which a gene produces a biochemical for example an RNA or polypeptide. It includes without limitation transcription of the gene into RNA molecules such as for example messenger RNA mRNA transfer RNA tRNA or any other RNA product and the translation of mRNA into polypeptide s .

Expression of a gene produces a gene product. As used herein a gene product can be either a nucleic acid e.g. a messenger RNA produced by transcription of a gene or a polypeptide which is translated from a transcript. Gene products described herein further include nucleic acids with post transcriptional modifications e.g. polyadenylation or polypeptides with post translational modifications e.g. methylation glycosylation the addition of lipids association with other protein subunits proteolytic cleavage et cetera.

As used herein the term gene refers to a polynucleotide comprising nucleotides that encode a functional molecule including functional molecules produced by transcription only e.g. a bioactive RNA species or by transcription and translation e.g. a polypeptide . The term gene encompasses cDNA and genomic DNA nucleic acids. Gene also refers to a nucleic acid fragment that expresses a specific RNA protein or polypeptide including regulatory sequences preceding 5 non coding sequences and following 3 non coding sequences the coding sequence. Native gene refers to a gene as found in nature with its own regulatory sequences. Chimeric gene refers to any gene that is not a native gene comprising regulatory and or coding sequences that are not found together in nature. Accordingly a chimeric gene may comprise regulatory sequences and coding sequences that are derived from different sources or regulatory sequences and coding sequences derived from the same source but arranged in a manner different than that found in nature. A chimeric gene may comprise coding sequences derived from different sources and or regulatory sequences derived from different sources. Endogenous gene refers to a native gene in its natural location in the genome of an organism. A foreign gene or heterologous gene refers to a gene not normally found in the host organism but that is introduced into the host organism by gene transfer. Foreign genes can comprise native genes inserted into a non native organism or chimeric genes. A transgene is a gene that has been introduced into the genome by a transformation procedure.

 RNA transcript refers to the product resulting from RNA polymerase catalyzed transcription of a DNA sequence. When the RNA transcript is a perfect complementary copy of the DNA sequence it is referred to as the primary transcript or it may be a RNA sequence derived from post transcriptional processing of the primary transcript and is referred to as the mature RNA. Messenger RNA mRNA refers to the RNA that is without introns and that can be translated into protein by the cell. cDNA refers to a double stranded DNA that is complementary to and derived from mRNA. Sense RNA refers to RNA transcript that includes the mRNA and so can be translated into protein by the cell. Antisense RNA refers to a RNA transcript that is complementary to all or part of a target primary transcript or mRNA and that blocks the expression of a target gene. The complementarity of an antisense RNA may be with any part of the specific gene transcript i.e. at the 5 non coding sequence 3 non coding sequence or the coding sequence.

A vector refers to any vehicle for the cloning of and or transfer of a nucleic acid into a host cell. A vector may be a replicon to which another DNA segment may be attached so as to bring about the replication of the attached segment. A replicon refers to any genetic element e.g. plasmid phage cosmid chromosome virus that functions as an autonomous unit of DNA replication in vivo i.e. capable of replication under its own control. The term vector includes both viral and nonviral vehicles for introducing the nucleic acid into a cell in vitro ex vivo or in vivo. A large number of vectors known in the art may be used to manipulate nucleic acids incorporate coding sequences into genes et cetera. Possible vectors include for example plasmids or modified viruses including for example bacteriophages such as lambda derivatives or plasmids such as pBR322 or pUC plasmid derivatives or the Bluescript vector. Another example of vectors that are useful in the present invention is the Ultra Vector Production System Intrexon Corp. Blacksburg Va. as described in WO 2007 038276 incorporated by reference herein. For example the insertion of the DNA fragments corresponding to response elements and promoters into a suitable vector for in vitro and or in vivo expression of modified forms of PE and fragments thereof as described herein including fusion proteins conjugates and otherwise linked forms can be accomplished by ligating the appropriate DNA fragments into a chosen vector that has complementary cohesive termini. Alternatively the ends of the DNA molecules may be enzymatically modified or any site may be produced by ligating nucleotide sequences linkers into the DNA termini. Such vectors may be engineered to contain selectable marker genes that provide for the selection of cells that have incorporated the marker into the cellular genome. Such markers allow identification and or selection of host cells that incorporate and express the proteins encoded by the marker.

Viral vectors and particularly retroviral vectors have been used in a wide variety of gene delivery applications in cells as well as living animal subjects. Viral vectors that can be used to express embodiments of the invention described herein include but are not limited to retrovirus adeno associated virus pox baculovirus vaccinia herpes simplex Epstein Barr adenovirus geminivirus and caulimovirus vectors. Non viral vectors include plasmids liposomes electrically charged lipids cytofectins DNA protein complexes and biopolymers. In addition to a nucleic acid a vector may also comprise one or more regulatory regions and or selectable markers useful in selecting measuring and monitoring nucleic acid transfer results e.g. monitoring transfer to target or non target tissues duration of expression et cetera .

The term plasmid refers to an extra chromosomal element often carrying a gene that is not part of the central metabolism of the cell and usually in the form of circular double stranded DNA molecules. Such elements may be autonomously replicating sequences genome integrating sequences phage or nucleotide sequences linear circular or supercoiled of a single or double stranded DNA or RNA derived from any source in which a number of nucleotide sequences have been joined or recombined into a unique construction which is capable of introducing a promoter fragment and DNA sequence for a selected gene product along with appropriate 3 untranslated sequence into a cell.

A cloning vector refers to a replicon which is a unit length of a nucleic acid preferably DNA that replicates sequentially and which comprises an origin of replication such as a plasmid phage or cosmid to which another nucleic acid segment may be attached so as to bring about the replication of the attached segment. Cloning vectors may be capable of replication in one cell type and expression in another shuttle vector . Cloning vectors may comprise one or more sequences that can be used for selection of cells comprising the vector and or one or more multiple cloning sites for insertion of sequences of interest. The term expression vector refers to a vector plasmid or vehicle designed to enable the expression of an inserted nucleic acid sequence following transformation into the host. The cloned gene i.e. the inserted nucleic acid sequence is usually placed under the control of control elements such as a promoter a minimal promoter an enhancer or the like. Initiation control regions or promoters which are useful to drive expression of a nucleic acid in the desired host cell are numerous and familiar to those skilled in the art. Virtually any promoter capable of driving expression of these genes can be used in an expression vector including but not limited to viral promoters bacterial promoters animal promoters mammalian promoters synthetic promoters constitutive promoters tissue specific promoters pathogenesis or disease related promoters developmental specific promoters inducible promoters light regulated promoters CYCl HIS3 GALl GAL4 GALlO ADHl PGK PH05 GAPDH ADCl TRPl URA3 LEU2 ENO TPI alkaline phosphatase promoters useful for expression in AOXl promoter useful for expression in 3 lactamase lac ara tet trp IP IP T7 tac and trc promoters useful for expression in light regulated seed specific pollen specific ovary specific cauliflower mosaic virus 35S CMV 35S minimal cassava vein mosaic virus CsVMV chlorophyll a b binding protein ribulose 1 5 bisphosphate carboxylase shoot specific root specific chitinase stress inducible rice tungro bacilliform virus plant super promoter potato leucine aminopeptidase nitrate reductase mannopine synthase nopaline synthase ubiquitin zein protein and anthocyanin promoters useful for expression in plant cells animal and mammalian promoters known in the art including but are not limited to the SV40 early SV40e promoter region the promoter contained in the 3 long terminal repeat LTR of Rous sarcoma virus RSV the promoters of the E1A or major late promoter MLP genes of adenoviruses Ad the cytomegalovirus CMV early promoter the herpes simplex virus HSV thymidine kinase TK promoter a baculovirus IE1 promoter an elongation factor 1 alpha EF1 promoter a phosphoglycerate kinase PGK promoter a ubiquitin Ubc promoter an albumin promoter the regulatory sequences of the mouse metallothionein L promoter and transcriptional control regions the ubiquitous promoters HPRT vimentin actin tubulin and the like the promoters of the intermediate filaments desmin neurofilaments keratin GFAP and the like the promoters of therapeutic genes of the MDR CFTR or factor VIII type and the like pathogenesis or disease related promoters and promoters that exhibit tissue specificity and have been utilized in transgenic animals such as the elastase I gene control region which is active in pancreatic acinar cells insulin gene control region active in pancreatic beta cells immunoglobulin gene control region active in lymphoid cells mouse mammary tumor virus control region active in testicular breast lymphoid and mast cells albumin gene Apo AI and Apo All control regions active in liver alpha fetoprotein gene control region active in liver alpha 1 antitrypsin gene control region active in the liver beta globin gene control region active in myeloid cells myelin basic protein gene control region active in oligodendrocyte cells in the brain myosin light chain 2 gene control region active in skeletal muscle and gonadotropic releasing hormone gene control region active in the hypothalamus pyruvate kinase promoter villin promoter promoter of the fatty acid binding intestinal protein promoter of the smooth muscle cell actin and the like. In addition these expression sequences may be modified by addition of enhancer or regulatory sequences and the like.

Vectors comprising polynucleotides of the invention may be introduced into the desired host cells by methods known in the art e.g. transfection electroporation microinjection transduction cell fusion DEAE dextran calcium phosphate precipitation lipofection lysosome fusion use of a gene gun or a DNA vector transporter see e.g. Wu et al . 267 963 1992 Wu et al . 263 14621 1988 and Hartmut et al Canadian Patent No. 2 012 311 .

Vectors and polynucleotides of the invention may be introduced in vivo by lipofection. For example via use of liposomes for encapsulation and transfection of nucleic acids in vitro. Synthetic cationic lipids designed to limit the difficulties encountered with liposome mediated transfection can be used to prepare liposomes for in vivo transfection of a gene encoding a marker Feigner et al USA. 84 7413 1987 Mackey et al USA 85 8027 1988 and Ulmer et al 259 1745 1993 . Use of cationic lipids may promote encapsulation of negatively charged nucleic acids and also promote fusion with negatively charged cell membranes Feigner et al 337 387 1989 . Particularly useful lipid compounds and compositions for transfer of nucleic acids are described in WO95 18863 WO96 17823 and U.S. Pat. No. 5 459 127.

Other molecules are also useful for facilitating transfection of a nucleic acid in vivo such as a cationic oligopeptide e.g. WO95 21931 peptides derived from DNA binding proteins e.g. WO96 25508 or a cationic polymer e.g. WO95 21931 .

It is also possible to introduce a vector in vivo as a naked DNA plasmid see U.S. Pat. Nos. 5 693 622 5 589 466 and 5 580 859 . Receptor mediated DNA delivery approaches can also be used Curiel et al. Hum. Gene Ther. 3 147 1992 and Wu et al. 262 4429 1987 .

The term transfection refers to the uptake of exogenous or heterologous RNA or DNA by a cell. A cell has been transfected by exogenous or heterologous RNA or DNA when such RNA or DNA has been introduced inside the cell.

 Transformation refers to the transfer of a nucleic acid fragment into the genome of a host organism resulting in genetically stable inheritance. Host organisms containing the transformed nucleic acid fragments are referred to as transgenic or recombinant or transformed organisms.

In addition recombinant vector comprising polynucleotides of the invention may include one or more origins for replication in the cellular hosts in which their amplification or their expression is sought markers or selectable markers.

The term selectable marker refers to an identifying factor usually an antibiotic or chemical resistance gene that is able to be selected for based upon the marker gene s effect i.e. resistance to an antibiotic resistance to a herbicide colorimetric markers enzymes fluorescent markers and the like wherein the effect is used to track the inheritance of a nucleic acid of interest and or to identify a cell or organism that has inherited the nucleic acid of interest. Examples of selectable marker genes known and used in the art include genes providing resistance to ampicillin streptomycin gentamycin kanamycin hygromycin bialaphos herbicide sulfonamide and the like and genes that are used as phenotypic markers i.e. anthocyanin regulatory genes isopentanyl transferase gene and the like.

The term reporter gene refers to a nucleic acid encoding an identifying factor that is able to be identified based upon the reporter gene s effect wherein the effect is used to track the inheritance of a nucleic acid of interest to identify a cell or organism that has inherited the nucleic acid of interest and or to measure gene expression induction or transcription. Examples of reporter genes known and used in the art include luciferase Luc green fluorescent protein GFP chloramphenicol acetyltransferase CAT galactosidase LacZ glucuronidase Gus and the like. Selectable marker genes may also be considered reporter genes.

 Promoter and promoter sequence are used interchangeably and refer to a DNA sequence capable of controlling the expression of a coding sequence or functional RNA. In general a coding sequence is located 3 to a promoter sequence. Promoters may be derived in their entirety from a native gene or be composed of different elements derived from different promoters found in nature or even comprise synthetic DNA segments. It is understood by those skilled in the art that different promoters may direct the expression of a gene in different tissues or cell types or at different stages of development or in response to different environmental or physiological conditions. Promoters that cause a gene to be expressed in most cell types at most times are commonly referred to as constitutive promoters. Promoters that cause a gene to be expressed in a specific cell type are commonly referred to as cell specific promoters or tissue specific promoters. Promoters that cause a gene to be expressed at a specific stage of development or cell differentiation are commonly referred to as developmentally specific promoters or cell differentiation specific promoters. Promoters that are induced and cause a gene to be expressed following exposure or treatment of the cell with an agent biological molecule chemical ligand light or the like that induces the promoter are commonly referred to as inducible promoters or regulatable promoters. It is further recognized that since in most cases the exact boundaries of regulatory sequences have not been completely defined DNA fragments of different lengths may have identical promoter activity.

The promoter sequence is typically bounded at its 3 terminus by the transcription initiation site and extends upstream 5 direction to include the minimum number of bases or elements necessary to initiate transcription at levels detectable above background. Within the promoter sequence will be found a transcription initiation site conveniently defined for example by mapping with nuclease Sl as well as protein binding domains consensus sequences responsible for the binding of RNA polymerase.

A coding sequence is under the control of transcriptional and translational control sequences in a cell when RNA polymerase transcribes the coding sequence into mRNA which is then trans RNA spliced if the coding sequence contains nitrons and translated into the protein encoded by the coding sequence.

 Transcriptional and translational control sequences refer to DNA regulatory sequences such as promoters enhancers terminators and the like that provide for the expression of a coding sequence in a host cell. In eukaryotic cells polyadenylation signals are control sequences.

The term response element RE refers to one or more cis acting DNA elements which confer responsiveness on a promoter mediated through interaction with the DNA binding domains of a transcription factor. This DNA element may be either palindromic perfect or imperfect in its sequence or composed of sequence motifs or half sites separated by a variable number of nucleotides. The half sites can be similar or identical and arranged as either direct or inverted repeats or as a single half site or multimers of adjacent half sites in tandem. The response element may comprise a minimal promoter isolated from different organisms depending upon the nature of the cell or organism into which the response element will be incorporated. The DNA binding domain of the transcription factor binds in the presence or absence of a ligand to the DNA sequence of a response element to initiate or suppress transcription of downstream gene s under the regulation of this response element.

Examples of DNA sequences for response elements of the natural ecdysone receptor include RRGG TTCANTGAC ACYY SEQ ID NO 140 see Cherbas et. al. Genes Dev. 5 120 131 1991 AGGTCAN n AGGTCA SEQ ID NO 141 where N n can be one or more spacer nucleotides see D Avino et al. Mol. Cell. Endocrinol. 113 1 1995 and GGGTTGAATGAATTT SEQ ID NO 142 see Antoniewski et al. Mol. Cell Biol. 14 4465 1994 .

The terms operably linked operably associated through operable association and the like refer to the association of nucleic acid sequences on a single nucleic acid fragment so that the function of one is affected by the other. For example a promoter is operably linked with a coding sequence when it is capable of affecting the expression of that coding sequence i.e. that the coding sequence is under the transcriptional control of the promoter . Coding sequences can be operably linked to regulatory sequences in sense or antisense orientation.

The terms cassette expression cassette and gene expression cassette refer to a segment of DNA that can be inserted into a nucleic acid or polynucleotide at specific restriction sites or by homologous recombination. The segment of DNA comprises a polynucleotide that encodes a polypeptide of interest and the cassette and restriction sites are designed to ensure insertion of the cassette in the proper reading frame for transcription and translation. Transformation cassette refers to a specific vector comprising a polynucleotide that encodes a polypeptide of interest and having elements in addition to the polynucleotide that facilitate transformation of a particular host cell. Cassettes expression cassettes gene expression cassettes and transformation cassettes of the invention may also comprise elements that allow for enhanced expression of a polynucleotide encoding a polypeptide of interest in a host cell. These elements may include but are not limited to a promoter a minimal promoter an enhancer a response element a terminator sequence a polyadenylation sequence and the like.

For purposes of expressing polynucleotides and polypeptides under control of a gene switch mechanism the term gene switch refers to the combination of a response element associated with a promoter and a ligand dependent transcription factor based system which in the presence of one or more ligands modulates the expression of a gene into which the response element and promoter are incorporated. Stated otherwise a gene switch refers to a peptide protein or polypeptide complex that functions to a bind an activating ligand and b regulate the transcription of a gene of interest in a ligand dependent fashion.

As used herein with respect to gene switch regulation systems the term dimerizes with the ligand binding domain that binds an activating ligand refers to a selective protein protein interaction that is induced by the presence of activating ligand.

As used herein the term ligand binding domain that binds an activating ligand refers to an amino acid sequence that selectively binds an activating ligand. In the methods disclosed herein an activating ligand binds to a ligand binding domain e.g. an ecdysone receptor ligand binding domain that is part of a ligand dependent transcriptional activation complex that regulates the expression of a polynucleotide sequence that encodes a gene of interest. Hence the expression of the gene of interest is regulated in a ligand dependent fashion.

The term ecdysone receptor based with respect to a gene switch refers to a gene switch comprising at least a functional part of a naturally occurring or synthetic ecdysone receptor ligand binding domain and which regulates gene expression in response to a ligand that binds to the ecdysone receptor ligand binding domain.

The terms modulate and modulates mean to induce reduce or inhibit nucleic acid or gene expression resulting in the respective induction reduction or inhibition of protein or polypeptide production.

Polynucleotides or vectors comprising sequences encoding polypeptides of the present invention may further comprise at least one promoter suitable for driving expression of a gene in a modified cell.

Enhancers that may be used in embodiments of the invention include but are not limited to an SV40 enhancer a cytomegalovirus CMV enhancer an elongation factor 1 EF1 enhancer yeast enhancers viral gene enhancers et cetera.

 Regulatory region refers to a nucleic acid sequence that regulates the expression of a second nucleic acid sequence. A regulatory region may include sequences which are naturally responsible for expressing a particular nucleic acid a homologous region or may include sequences of a different origin that are responsible for expressing different proteins or even synthetic proteins a heterologous region . In particular the sequences can be sequences of prokaryotic eukaryotic or viral genes or derived sequences that stimulate or repress transcription of a gene in a specific or non specific manner and in an inducible or non inducible manner. Regulatory regions include origins of replication RNA splice sites promoters enhancers transcriptional termination sequences and signal sequences which direct the polypeptide into the secretory pathways of the target cell.

The term exogenous gene or heterologous gene means a gene foreign to the subject or organism that is a gene which is introduced into the subject through a transformation process an unmutated version of an endogenous mutated gene or a mutated version of an endogenous unmutated gene. The method of transformation is not critical to this invention and may be any method suitable for the subject known to those in the art. Exogenous genes can be either natural or synthetic genes which are introduced into the subject in the form of DNA or RNA which may function through a DNA intermediate such as by reverse transcriptase. Such genes can be introduced into target cells directly introduced into the subject or indirectly introduced by the transfer of transformed cells into the subject.

Polynucleotides and polypeptides of the invention may be expressed in vivo under control of a gene switch control mechanism such as those described in for example but not limited to 

The term ligand dependent transcription factor complex or LDTFC refers to a transcription factor comprising one or more protein subunits which complex can regulate gene expression driven by a factor regulated promoter as defined herein. A model LDTFC is an ecdysone receptor complex generally refers to a heterodimeric protein complex having at least two members of the nuclear receptor family ecdysone receptor EcR and ultraspiracle USP proteins see Yao et al. 366 476 1993 Yao et al. 71 63 1992 . A functional LDTFC such as an EcR complex may also include additional protein s such as immunophilins. Additional members of the nuclear receptor family of proteins known as transcriptional factors such as DHR38 betaFTZ 1 or other insect homologs may also be ligand dependent or independent partners for EcR and or USP. A LDTFC such as an EcR complex can also be a heterodimer of EcR protein and the vertebrate homolog of ultraspiracle protein retinoic acid X receptor RXR protein or a chimera of USP and RXR. The terms LDTFC and EcR complex also encompass homodimer complexes of the EcR protein or USP as well as single polypeptides or trimers tetramer and other multimers serving the same function.

A LDTFC such as an EcR complex can be activated by an active ecdysteroid or non steroidal ligand bound to one of the proteins of the complex inclusive of EcR but not excluding other proteins of the complex. As used herein the term ligand as applied to LDTFC based gene switches e.g. EcD complex based gene switches describes small and soluble molecules having the capability of activating a gene switch to stimulate expression of a polypeptide encoded therein. Examples of ligands include without limitation an ecdysteroid such as ecdysone 20 hydroxyecdysone ponasterone A muristerone A and the like 9 cis retinoic acid synthetic analogs of retinoic acid N N diacylhydrazines such as those disclosed in U.S. Pat. Nos. 6 013 836 5 117 057 5 530 028 5 378 726 and 7 304 161 and U.S. Pat. No. 7 456 315 oxadiazolines as described in U.S. Pat. No. 7 304 162 dibenzoylalkyl cyanohydrazines such as those disclosed in European Patent No. 461 809B1 N alkyl N N diaroylhydrazines such as those disclosed in U.S. Pat. No. 5 225 443 N acyl N alkylcarbonylhydrazines such as those disclosed in European Patent No. 234 994B1 N aroyl N alkyl N aroylhydrazines such as those described in U.S. Pat. No. 4 985 461 amidoketones such as those described in U.S. Pat. No. 7 375 093 each of which is incorporated herein by reference and other similar materials including 3 5 di tert butyl 4 hydroxy N isobutyl benzamide 8 O acetylharpagide oxysterol s 22 R hydroxycholesterol 24 S hydroxycholesterol 25 epoxycholesterol T0901317 5 alpha 6 alpha epoxycholesterol 3 sulfate ECHS 7 ketocholesterol 3 sulfate famesol bile acids 1 1 biphosphonate esters juvenile hormone III and the like. Examples of diacylhydrazine ligands useful in the present invention include RG 115819 3 5 Dimethyl benzoic acid N 1 ethyl 2 2 dimethyl propyl N 2 methyl 3 methoxy benzoyl hydrazide RG 115932 R 3 5 Dimethyl benzoic acid N 1 tert butyl butyl N 2 ethyl 3 methoxy benzoyl hydrazide and RG 115830 3 5 Dimethyl benzoic acid N 1 tert butyl butyl N 2 ethyl 3 methoxy benzoyl hydrazide . See e.g. U.S. Pat. No. 8 076 517 Publication No. 2009 0163592 and PCT Appl. No. PCT US2008 006757 WO 2008 153801 both of which are incorporated herein by reference in their entireties.

A LDTFC such as an EcR complex includes proteins which are members of the nuclear receptor superfamily wherein all members are characterized by the presence of one or more polypeptide subunits comprising an amino terminal transactivation domain AD TD or TA used interchangeably herein a DNA binding domain DBD and a ligand binding domain LBD . The AD may be present as a fusion with a heterodimerization partner or HP. A fusion protein comprising an AD and HP of the invention is referred to herein as a coactivation protein or CAP. The DBD and LBD may be expressed as a fusion protein referred to herein as a ligand inducible transcription factor LTF . The fusion partners may be separated by a linker e.g. a hinge region. Some members of the LTF family may also have another transactivation domain on the carboxy terminal side of the LBD. The DBD is characterized by the presence of two cysteine zinc fingers between which are two amino acid motifs the P box and the D box which confer specificity for ecdysone response elements. These domains may be either native modified or chimeras of different domains of heterologous receptor proteins.

EcR ligands when used with a LDTFC e.g. an EcR complex which in turn is bound to the response element linked to an exogenous gene e.g. a reporter gene provide the means for external temporal regulation of expression of the exogenous gene. The order in which the various components bind to each other that is ligand to receptor complex and receptor complex to response element is not critical. Typically modulation of expression of the exogenous gene is in response to the binding of a LDTFC e.g. an EcR complex to a specific control or regulatory DNA element. The EcR protein like other members of the nuclear receptor family possesses at least three domains an AD a DBD and a LBD. This receptor like a subset of the nuclear receptor family also possesses less well defined regions responsible for heterodimerization properties referred to herein as a heterodimerization partner or HP . Binding of the ligand to the ligand binding domain of a LTF e.g. an EcR protein after heterodimerization with a CAP including e.g. an AD and or an HP e.g. a USP or RXR protein enables the DNA binding domains of the heterodimeric proteins to bind to the response element in an activated form thus resulting in expression or suppression of the exogenous gene. This mechanism does not exclude the potential for ligand binding to individual subunits e.g. LTF or CAP e.g. an EcR or USP and the resulting formation of active homodimer complexes e.g. EcR EcR or USP USP . In one embodiment one or more of the receptor domains can be varied producing a chimeric gene switch. Typically one or more of the three domains may be chosen from a source different than the source of the other domains so that the chimeric receptor is optimized in the chosen host cell or organism for transactivating activity complementary binding of the ligand and recognition of a specific response element. In addition the response element itself can be modified or substituted with response elements for other DNA binding protein domains such as the GAL 4 protein from yeast see Sadowski et al. 335 563 1988 or LexA protein from see Brent et al. 43 729 736 1985 to accommodate chimeric LDTFCs e.g. EcR complexes. Another advantage of chimeric systems is that they allow choice of a promoter used to drive the exogenous gene according to a desired end result. Such double control can be particularly important in areas of gene therapy especially when cytotoxic proteins are produced because both the timing of expression as well as the cells wherein expression occurs can be controlled. When exogenous genes operatively linked to a suitable promoter are introduced into the cells of the subject expression of the exogenous genes is controlled by the presence of the ligand of this invention. Promoters may be constitutively or inducibly regulated or may be tissue specific that is expressed only in a particular type of cell or specific to certain developmental stages of the organism.

For in vivo use the ligands described herein may be taken up in pharmaceutically acceptable carriers such as for example solutions suspensions tablets capsules ointments elixirs and injectable compositions. Pharmaceutical compositions may contain from 0.01 to 99 by weight of the ligand. Compositions may be either in single or multiple dose forms. The amount of ligand in any particular pharmaceutical composition will depend upon the effective dose that is the dose required to elicit the desired gene expression or suppression.

Suitable routes of administering the pharmaceutical preparations include oral rectal topical including dermal buccal and sublingual vaginal parenteral including subcutaneous intramuscular intravenous intradermal intrathecal and epidural and by naso gastric tube. It will be understood by those skilled in the art that the preferred route of administration will depend upon the condition being treated and may vary with factors such as the condition of the recipient.

As used herein the terms treat or treatment refer to both therapeutic treatment and prophylactic or preventative measures wherein the object is to prevent or slow down lessen an undesired physiological change or disorder such as the development progression or spread i.e. metastasis of cancer. Beneficial or desired clinical results include but are not limited to alleviation of symptoms diminishment of extent of disease stabilized i.e. not worsening state of disease delay or slowing of disease progression amelioration or palliation of the disease state and remission whether partial or total . Treatment can also mean prolonging survival as compared to expected survival if not receiving treatment. Those in need of treatment include those already with the condition or disorder as well as those prone to have the condition or disorder or those in which the condition or disorder is to be prevented.

The terms subject individual animal patient or mammal is meant any subject particularly a mammalian subject for whom diagnosis prognosis or therapy is desired. Mammalian subjects include without limitation humans domestic animals farm animals and zoo sports or pet animals such as dogs cats guinea pigs rabbits rats mice horses cattle cows et cetera.

The terms hyperproliferative disease or disorder is intended to encompass all neoplastic cell growth and proliferation whether malignant or benign including all transformed cells and tissues and all cancerous cells and tissues. Hyperproliferative diseases or disorders include but are not limited to precancerous lesions abnormal cell growths tumors whether benign or malignant cancer and other hyperplasias.

The term cancer includes but is not limited to primary malignant cells or tumors e.g. those whose cells have not migrated to sites in the subject s body other than the site of the original malignancy or tumor and secondary malignant cells or tumors e.g. those arising from metastasis the migration of malignant cells or tumor cells to secondary sites that are different from the site of the original tumor .

A tumor or tumor tissue may also comprise tumor associated non tumor cells e.g. vascular cells which form blood vessels to supply the tumor or tumor tissue. Non tumor cells may be induced to replicate and develop by tumor cells for example the induction of angiogenesis in a tumor or tumor tissue.

Some examples of cancer include but are not limited to carcinoma lymphoma blastoma sarcoma and leukemia or lymphoid malignancies. More particular examples of such cancers are noted below and include squamous cell cancer e.g. epithelial squamous cell cancer lung cancer including small cell lung cancer non small cell lung cancer adenocarcinoma of the lung and squamous carcinoma of the lung cancer of the peritoneum hepatocellular cancer gastric or stomach cancer including gastrointestinal cancer pancreatic cancer glioblastoma cervical cancer ovarian cancer liver cancer bladder cancer hepatoma breast cancer colon cancer rectal cancer colorectal cancer endometrial cancer or uterine carcinoma salivary gland carcinoma kidney or renal cancer prostate cancer vulval cancer thyroid cancer hepatic carcinoma as well as head e.g. brain and neck cancer.

Other examples of cancers or malignancies include but are not limited to Acute Childhood Lymphoblastic Leukemia Acute Lymphoblastic Leukemia Acute Lymphocytic Leukemia Acute Myeloid Leukemia Adrenocortical Carcinoma Adult Primary Hepatocellular Cancer Adult Primary Liver Cancer Adult Acute Lymphocytic Leukemia Adult Acute Myeloid Leukemia Adult Hodgkin s Disease Adult Hodgkin s Lymphoma Adult Lymphocytic Leukemia Adult Non Hodgkin s Lymphoma Adult Primary Liver Cancer Adult Soft Tissue Sarcoma AIDS Related Lymphoma AIDS Related Malignancies Anal Cancer Astrocytoma Bile Duct Cancer Bladder Cancer Bone Cancer Brain Stem Glioma Brain Tumors Breast Cancer Cancer of the Renal Pelvis and Ureter Central Nervous System Primary Lymphoma Central Nervous System Lymphoma Cerebellar Astrocytoma Cerebral Astrocytoma Cervical Cancer Childhood Primary Hepatocellular Cancer Childhood Primary Liver Cancer Childhood Acute Lymphoblastic Leukemia Childhood Acute Myeloid Leukemia Childhood Brain Stem Glioma Childhood Cerebellar Astrocytoma Childhood Cerebral Astrocytoma Childhood Extracranial Germ Cell Tumors Childhood Hodgkin s Disease Childhood Hodgkin s Lymphoma Childhood Hypothalamic and Visual Pathway Glioma Childhood Lymphoblastic Leukemia Childhood Medulloblastoma Childhood Non Hodgkin s Lymphoma Childhood Pineal and Supratentorial Primitive Neuroectodermal Tumors Childhood Primary Liver Cancer Childhood Rhabdomyosarcoma Childhood Soft Tissue Sarcoma Childhood Visual Pathway and Hypothalamic Glioma Chronic Lymphocytic Leukemia Chronic Myelogenous Leukemia Colon Cancer Cutaneous T cell Lymphoma Endocrine Pancreas Islet Cell Carcinoma Endometrial Cancer Ependymoma Epithelial Cancer Esophageal Cancer Ewing s Sarcoma and Related Tumors Exocrine Pancreatic Cancer Extracranial Germ Cell Tumor Extragonadal Germ Cell Tumor Extrahepatic Bile Duct Cancer Eye Cancer Female Breast Cancer Gaucher s Disease Gallbladder Cancer Gastric Cancer Gastrointestinal Carcinoid Tumor Gastrointestinal Tumors Germ Cell Tumors Gestational Trophoblastic Tumor Hairy Cell Leukemia Head and Neck Cancer Hepatocellular Cancer Hodgkin s Disease Hodgkin s Lymphoma Hypergammaglobulinemia Hypopharyngeal Cancer Intestinal Cancers Intraocular Melanoma Islet Cell Carcinoma Islet Cell Pancreatic Cancer Kaposi s Sarcoma Kidney Cancer Laryngeal Cancer Lip and Oral Cavity Cancer Liver Cancer Lung Cancer Lymphoproliferative Disorders Macroglobulinemia Male Breast Cancer Malignant Mesothelioma Malignant Thymoma Medulloblastoma Melanoma Mesothelioma Metastatic Occult Primary Squamous Neck Cancer Metastatic Primary Squamous Neck Cancer Metastatic Squamous Neck Cancer Multiple Myeloma Multiple Myeloma Plasma Cell Neoplasm Myelodysplastic Syndrome Myelogenous Leukemia Myeloid Leukemia Myeloproliferative Disorders Nasal Cavity and Paranasal Sinus Cancer Nasopharyngeal Cancer Neuroblastoma Non Hodgkin s Lymphoma During Pregnancy Nonmelanoma Skin Cancer Non Small Cell Lung Cancer Occult Primary Metastatic Squamous Neck Cancer Oropharyngeal Cancer Osteo Malignant Fibrous Sarcoma Osteosarcoma Malignant Fibrous Histiocytoma Osteosarcoma Malignant Fibrous Histiocytoma of Bone Ovarian Epithelial Cancer Ovarian Germ Cell Tumor Ovarian Low Malignant Potential Tumor Pancreatic Cancer Paraproteinemias Purpura Parathyroid Cancer Pheochromocytoma Pituitary Tumor Plasma Cell Neoplasm Multiple Myeloma Primary Central Nervous System Lymphoma Primary Liver Cancer Prostate Cancer Rectal Cancer Renal Cell Cancer Renal Pelvis and Ureter Cancer Retinoblastoma Rhabdomyosarcoma Salivary Gland Cancer Sarcoidosis Sarcomas Sezary Syndrome Skin Cancer Small Cell Lung Cancer Small Intestine Cancer Soft Tissue Sarcoma Squamous Neck Cancer Stomach Cancer Supratentorial Primitive Neuroectodermal and Pineal Tumors T cell Lymphoma Testicular Cancer Thymoma Thyroid Cancer Transitional Cell Cancer of the Renal Pelvis and Ureter Transitional Renal Pelvis and Ureter Cancer Trophoblastic Tumors Ureter and Renal Pelvis Cell Cancer Urethral Cancer Uterine Cancer Uterine Sarcoma Vaginal Cancer Visual Pathway and Hypothalamic Glioma Vulvar Cancer Waldenstrom s Macroglobulinemia Wilms Tumor and any other hyperproliferative disease besides neoplasia located in an organ system listed above.

Polypeptides may be made to differ by introduction of conservative or non conservative amino acid changes. Conservative amino acid substitutions refer to the interchangeability of residues having similar amino acid side chains. Conservative amino acid substitutions refer to substitutions of one or more amino acids in a native amino acid sequence e.g. wild type or naturally occurring form of PE with other amino acid s having similar side chains e.g. side chains similar in terms of size charge element composition and or hydrophobicity hydrophilicity .

Conserved substitutes for an amino acid within a native amino acid sequence can be selected from other members of the group to which the naturally occurring amino acid belongs. For example conservative amino acid residue substitution groups include 

Other substitution groups of amino acids can be envisioned. For example amino acids can be grouped by similar function or chemical structure or composition e.g. acidic basic aliphatic aromatic sulfur containing . For example an Aliphatic grouping may comprise Glycine G Alanine A Valine V Leucine L Isoleucine I . Other groups containing amino acids that are considered conservative substitutions for one another include 

Exemplary embodiments of conservative amino acid substitutions include the interchangeability of valine leucine valine isoleucine leucine phenylalanine tyrosine lysine arginine alanine valine aspartic acid glutamic acid and asparagine glutamine.

Examples of a few of the many possible amino acid analogs routinely known to those of skill in the art include for example but without limitation analogs such as 4 hydroxyproline which may be substituted for proline 5 hydroxylysine which may be substituted for lysine 3 methylhistidine which may be substituted for histidine homoserine which may be substituted for serine and ornithine which may be substituted for lysine.

Examples of a few of the many possible non standard amino acids routinely known to those of skill in the art include for example but without limitation molecules such as ornithine citrulline lanthionine 2 aminoisobutyric acid dehydroalanine aminobutyric acid alanine 3 aminopropanoic acid selenocysteine and pyrrolysine.

Substitution mutations may be made by any technique for mutagenesis known in the art including for example but not limited to in vitro site directed mutagenesis Hutchinson et al 255 6551 1978 Zoller et al 3 479 1984 Oliphant et al 44 177 1986 Hutchinson et al 83 710 1986 use of TAB linkers Pharmacia restriction endonuclease digestion fragment deletion and substitution PCR mediated oligonucleotide directed mutagenesis et cetera. PCR based techniques are preferred for site directed mutagenesis see Higuchi 1989 Using PCR to Engineer DNA in PCR Technology Principles and Applications for DNA Amplification H. Erlich ed. Stockton Press Chapter 6 pp. 61 70 .

Embodiments of the invention include isolated polypeptides proteins comprising or consisting of a modified form of exotoxin A or a fragment thereof wherein said modified form or fragment thereof comprises an epitope selected from the group consisting of 

Embodiments of the invention include isolated polypeptides proteins comprising or consisting of a modified form of exotoxin A or a fragment thereof wherein said modified form or fragment thereof comprises an epitope selected from the group consisting of 

Embodiments of the invention include isolated polypeptides proteins comprising or consisting of a modified form of exotoxin A or a fragment thereof wherein said modified form or fragment thereof comprises an epitope selected from the group consisting of 

Embodiments of the invention include isolated polypeptides proteins comprising or consisting of a modified form of exotoxin A or a fragment thereof wherein said modified form or fragment thereof comprises an epitope selected from the group consisting of 

Embodiments of the invention include isolated polypeptides proteins comprising or consisting of a modified form of exotoxin A or a fragment thereof wherein said modified form or fragment thereof comprises an epitope selected from the group consisting of 

Embodiments of the invention also comprise or consist of isolated polypeptides and peptides comprising or consisting of the above referenced amino acids sequences except wherein one or more amino acids have been substituted with conservative amino acids substitutions. Embodiments of the invention further comprise or consist of isolated polypeptides proteins and peptides comprising or consisting of the above referenced amino acids sequences except wherein one or more amino acids have been substituted with amino acids which are naturally occurring non naturally occurring non standard amino acids or amino acid analogs.

Embodiments of the invention include isolated polypeptides proteins comprising or consisting of a modified form of exotoxin A or a fragment thereof wherein said modified form or fragment thereof comprises an epitope selected from the group consisting of 

Embodiments of the invention include an isolated polypeptide comprising a modified form of exotoxin A or a fragment thereof wherein said modified form or fragment thereof comprises one or more amino acid substitutions selected from the group consisting of 

Embodiments of the invention include an isolated polypeptide comprising a modified form of exotoxin A or a fragment thereof wherein said modified form or fragment thereof comprises one or more amino acid substitutions selected from the group consisting of 

Embodiments of the invention comprise isolated polypeptides as described above including polypeptides comprising amino acid substitutions introduced at each of amino acid positions 141 146 149 150 152 184 189 194 197 233 234 241 321 324 326 and 327 in comparison to the amino acid sequence of SEQ ID NO 1 .

Embodiments of the invention include isolated polypeptides proteins and peptides comprising or consisting of the following amino acid sequences 

Embodiments of the invention also comprise or consist of isolated polypeptides proteins and peptides comprising or consisting of the above referenced amino acids sequences except wherein one or more amino acids have been substituted with conservative amino acids substitutions. Embodiments of the invention also comprise or consist of isolated polypeptides proteins and peptides comprising or consisting of the above referenced amino acids sequences except wherein one or more amino acids have been substituted with amino acids which are naturally occurring non naturally occurring non standard amino acids or amino acid analogs.

Embodiments of the invention include polypeptides comprising a PE A Domain III i.e. a cytotoxic domain see e.g. . Examples of sequences comprising a cytotoxic portion of PE can be found in SEQ ID NO 1 and SEQ ID NO 4 spanning amino acid residues Phe 134 to Lys 347. Examples of sequences comprising a cytotoxic portion of PE can also be found in SEQ ID NO 133 and SEQ ID NO 134 spanning amino acid residues Phe 400 to Lys 613.

Embodiments of the invention include polypeptides comprising a PE A Domain III i.e. a cytotoxic domain and one or more PE A domains selected from the group consisting of 

Embodiments of the invention further comprise PE variants. For example such variants include without limitation PE polypeptide examples as shown in SEQ ID Nos 143 to 163 and SEQ ID No 175.

Embodiments of the invention comprise any one or more of the PE A domains indicated in the preceding paragraphs wherein said one or more domains are chemically linked covalently coupled or fused i.e. as in frame fusion proteins with a heterologous polypeptide for example such as ligand or antigen binding polypeptide .

Embodiments of the invention include polypeptides comprising PE wherein one or more amino acids are substituted with any combination of one or more conservative amino acid substitutions non conservative amino acid substitutions non naturally occurring amino acid substitutions non standard amino acids and or substitutions with amino acid analogs and further wherein said polypeptides are non immunogenic or exhibit reduced immunogenicity as determined and assayed by comparison to immunogenicity of corresponding non amino acid substituted forms of PE as measured using in vitro or in vivo assays. In particular embodiments amino acid substituted forms of PE are at least 25 at least about 25 at least 50 at least about 50 at least 75 or at least about 75 less immunogenic compared to corresponding non amino acid substituted forms of PE. In particular embodiments amino acid substituted forms of PE are at least 2 fold at least about 2 fold at least 3 fold at least about 3 fold at least 4 fold at least about 4 fold at least 5 fold at least about 5 fold at least 10 fold at least about 10 fold at least 50 fold at least about 50 fold at least 100 fold at least about 100 fold at least 500 fold at least about 500 fold at least 1000 fold or at least about 1000 fold less immunogenic compared to corresponding non amino acid substituted forms of PE. In one embodiment amino acid substituted forms of PE are non immunogenic or exhibit undetectable immunogenicity compared to corresponding non amino acid substituted forms of PE.

The immunogenicity of substituted peptides may be measured via assays routinely known and used by those of skill in the art. For example immunogenicity may be assayed by methods including but not limited to the proliferation assays described in Example 1 herein.

Additionally methods for predicting and assays for assessing immunogenicity include those methods and assays such as described or referenced in 

Embodiments of the invention include polypeptides comprising PE wherein one or more amino acids are substituted with any combination of one or more conservative amino acid substitutions non conservative amino acid substitutions non naturally occurring amino acid substitutions non standard amino acids and or substitutions with amino acid analogs and further wherein said polypeptides retain biological activity as determined and assayed by comparison to biological activities of corresponding non amino acid substituted forms of PE such as but not limited to cell killing activity cell cytotoxicity inactivation of the translation elongation factor EF 2 ADP ribosylation of EF 2 and inhibition of protein synthesis as measured using in vitro or in vivo assays. In particular embodiments amino acid substituted forms of PE exhibit 100 or about 100 of biological activity compared to corresponding non amino acid substituted forms of PE. In particular embodiments amino acid substituted forms of PE exhibit at least 95 or at least about 95 of biological activity compared to corresponding non amino acid substituted forms of PE. In particular embodiments amino acid substituted forms of PE exhibit at least 90 at least about 90 at least 85 at least about 85 at least 80 at least about 80 at least 75 at least about 75 at least 70 at least about 70 at least 60 at least about 60 at least 50 or at least about 50 of biological activity compared to corresponding non amino acid substituted forms of PE.

Embodiments of the invention further comprise fusion proteins conjugates covalently linked and non covalently linked amino acid substituted forms of PE or fragments thereof as described herein. Amino acid substituted forms of PE may be fused conjugated or otherwise linked with any artificial recombinant or naturally occurring molecule or polypeptide to modify PE activity and or PE localization targeting such as by conferring to PE via said fusion or conjugation the tissue targeting cell targeting or sub cellular localization properties of the molecule to which PE is fused conjugated or otherwise linked. For example but without limitation amino acid substituted forms or fragments thereof of PE may be fused conjugated or otherwise linked with any type of antibody or antigen binding fragments thereof cell surface receptor secreted or cell surface ligand or fragments thereof.

In one embodiment amino acid substituted forms of PE including amino acid substituted forms of PE fused conjugated or otherwise linked to another molecule or polypeptide are useful in the treatment of cancer including but not limited to types of cancer described herein. In one embodiment amino acid substituted forms of PE as described herein are useful for the preparation of a medicament for the treatment of cancer including but not limited to types of cancer described herein.

In one embodiment amino acid substituted forms of PE or fragments thereof may be fused conjugated or otherwise linked without limitation antigen binding moieties such as antibodies or fragments thereof which specifically or preferentially bind to disease associated antigens. Such molecules include for example but without limitation antibodies indicated in Table 1.

In certain embodiments amino acid substituted forms of PE or fragments thereof may be fused conjugated or otherwise linked without limitation to naturally occurring normal or disease related molecules such as secreted extracellular intracellular transmembrane or cell surface bound molecules or fragments thereof or non naturally occurring variants and fragments thereof such as without limitation ligands receptors receptor extracellular domains cytokines growth factors cell signaling proteins extracellular and intracellular enzymes structural proteins cell adhesion proteins and molecules cluster of differentiation CD molecules mitogens cell division regulating molecules cancer tumor markers and antigens et cetera. In certain embodiments molecules which are normally transmembrane and cell surface bound polypeptides may be fused or conjugated to amino acid substituted forms of PE as polypeptide fragments lacking at least their transmembrane domains or polypeptide regions responsible for cell surface binding.

In certain embodiments molecules may be fused or conjugated to amino acid substituted forms of PE wherein such molecules possess or retain the ability even as fusion proteins or protein conjugates to form multimeric complexes such as hetero and homopolymers including but not limited to dimers trimers tetramers pentamers hexamers et cetera. 

In certain embodiments amino acid substituted forms of PE or fragments thereof may be generated as in frame polypeptide fusion proteins with molecules such as but not limited to those referenced above wherein the PE moiety is either an amino terminal portion or a carboxyl terminal portion of the fusion protein. Determination of which of these two configurations provides the desired results and or biological activities may be determined by routine experimentation practiced by those skilled in the art.

In certain embodiments amino acid substituted forms of PE or fragments thereof may be generated as fusion proteins wherein heterologous amino acid sequences such as cell targeting sequences are inserted within the amino acid substituted form of PE i.e. heterologous amino acids are flanked at the amino terminus and at the carboxy terminus by PE amino acid sequences . An example of a non amino acid substituted form of PE in such a configuration is demonstrated in U.S. Pat. No. 8 854 044 wherein a TGF polypeptide is incorporated at amino acid residues 607 to 604 within a PE37 polypeptide sequence. See e.g. U.S. Pat. No. 8 854 044 FIG. 1.

Some examples of molecules which may be fused conjugated or otherwise linked to amino acid substituted forms of PE include for example but without limitation those such as indicated in Table 2.

In one embodiment the present invention includes isolated nucleic acids and methods of expressing nucleic acids encoding any of the herein referenced modified forms of PE including fusions conjugates and otherwise linked molecules whether such forms are expressed from a single or one more separate polynucleotide sequences whether such polynucleotide sequences are expressed from a single or one or more separate expression vectors.

In one embodiment the present invention includes methods of making and using recombinant expression vectors to express nucleic acids encoding polypeptides comprising any of the herein referenced modified forms of PE including fusions conjugates and otherwise linked molecules. Use of a wide variety of expression vectors are well known and routinely used by those skilled in the art. A few examples of the types of expression vectors which may be used include but are not limited to derivatives of human or animal viruses such as retrovirus adeno associated virus pox baculovirus vaccinia herpes simplex Epstein Barr adenovirus geminivirus and caulimovirus vectors and insect viruses such as baculovirus yeast vectors bacteriophage vectors e.g. bacteriophage lambda plasmids cosmids artificial chromosomes liposomes electrically charged lipids cytofectins DNA protein complexes and biopolymers.

A wide variety of methods i.e. gene delivery systems are available and well known to those of skill in the art any of such methods may be used for introducing nucleic acids encoding modified forms of PE into a cell tissue or organism for in vitro in vivo in situ or ex vivo expression. The methods referenced below represent examples of ways in which nucleic acid s encoding modified forms of PE may be introduced into a cell. These examples are in no way intended to limit the scope of that may be used for gene delivery and expression of modified forms of PE in cells tissues or organisms these examples are presented to illustrate the many available methods.

Gene therapy based methods can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . As one example polynucleotides operably encoding the target nucleic acid can be delivered to a tissue or organism either as naked nucleic acid or as part of an expression vector. The term vector includes for example but is not limited to vectors such as plasmid vectors cosmid vectors artificial chromosome vectors and viral vectors. Some examples of viral vectors include adenovirus herpes simplex virus HSV alphavirus simian virus 40 picomavirus vaccinia virus retrovirus lentivirus and adeno associated virus. Vectors encoding modified forms of PE may be capable of replication in a cell in which it is introduced or it may be preferred that the vector is not capable of replication. Vectors encoding modified forms of PE may be capable of integration into the genomic DNA of a cell and subsequent expression therefrom or it may be preferred that the vector is not capable of integrating into the host genome. An example of a vector that can integrate into the genomic DNA of a cell is a retroviral vector in which an integrase enzyme mediates integration of the retroviral vector sequences. A vector may also contain transposon sequences that facilitate integration of the coding region into the genomic DNA of a host cell. Liposomes represent another manner in which target DNA may be delivered to a subject.

Selection of a vector depends upon a variety of desired characteristics in the resulting construct such as a selection marker vector replication rate type of target host cell species of host organism desired duration of protein expression. An expression vector optionally includes expression control sequences operably linked to the coding sequence such that the coding region is expressed in the cell. The invention is not limited by the use of any particular promoter and a wide variety is known. Promoters act as regulatory signals that bind RNA polymerase in a cell to initiate transcription of a downstream 3 direction operably linked coding sequence. The promoter used in the invention may be a constitutive or an inducible promoter. It can be but need not be heterologous with respect to the cell to which it is introduced.

In certain embodiments adenovirus expression vectors can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . The terms adenovirus expression vector is meant to include those constructs containing nucleic acid sequences sufficient to a support packaging of the construct and b to ultimately express a recombinant gene construct that has been inserted therein. In contrast to retroviruses use of adenovirus vectors does not result in chromosomal integration because adenovirus DNA replicates in an episomal manner. Moreover adenoviruses are considered to be structurally stable with no genome rearrangement occurring even after extensive virus reproduction and amplification. Methods of constructing and using adenovirus vectors as gene delivery systems are well known to those of skill in the art.

In certain embodiments adeno associated virus AAV expression vectors can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . AAV may be desirable for a number or reasons for example because AAV vectors exhibit a high frequency of integration can infect nondividing cells and have a broad host range. AAV is a dependent parvovirus in that it requires coinfection with another virus either adenovirus or a member of the herpes virus family to undergo a productive infection in cultured cells. In the absence of coinfection with helper virus the wild type AAV genome integrates through its ends into a human chromosome where it resides as a latent provirus. When a cell containing latent AAV provirus is superinfected with a helper virus the AAV genome is rescued from the chromosome and a normal productive infection is established. Methods of constructing and using AAV vectors as gene delivery systems are well known to those of skill in the art.

In certain embodiments retrovirus expression vectors can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . Retroviruses are a group of single stranded RNA viruses characterized by the ability to convert their genomic RNA to double stranded DNA in infected cells through a reverse transcription process. The resulting DNA stably integrates into cellular chromosomes as a provirus and directs synthesis of viral proteins. Retroviral integration results in the retention of viral gene sequences in the recipient cell and in its descendants. Retroviral vectors are able to infect a broad variety of cell types. Methods of constructing and using retroviruses as gene delivery systems are well known to those of skill in the art.

Many other expression vectors can also be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . For example vectors derived from viruses such as vaccinia viruses herpesviruses equine encephalitis viruses hepatitis viruses and lentiviruses can be used. Methods of constructing and using viral expression vectors as gene delivery systems are well known to those of skill in the art. The examples of such vectors referenced herein are not intended to be limiting with respect to the means by which modified forms of PE may be delivered and expressed in various host cells tissues or organisms.

In addition to viral delivery of modified target nucleic acid the following are additional methods of recombinant gene delivery can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . Methods of constructing and using non viral gene delivery systems are well known to those of skill in the art. See for example Al Dosari et al. Nonviral gene delivery principle limitations and recent progress 11 4 671 681 2009 and references cited therein.

In certain embodiments electroporation can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . Methods of using electroporation are well known to those of skill in the art. See for example Bodles Brakhop et al. Electroporation for the delivery of DNA based vaccines and immunotherapeutics current clinical developments 17 4 585 592 2009 and references cited therein. See also Golzio et al. Observations of the mechanisms of electromediated DNA uptake from vesicles to tissues 10 4 256 266 2010 and references cited therein. See also Andre et al. Nucleic acids electrotransfer in vivo mechanisms and practical aspects 10 4 267 280 2010 and references cited therein. See also Wells Electroporation and ultrasound enhanced non viral gene delivery in vitro and in vivo 26 1 21 28 2010 and references cited therein.

In certain embodiments particle bombardment can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . This method depends on the ability to accelerate nucleic acid coated microprojectiles to a sufficient velocity to allow them to pierce cell membranes thereby delivering nucleic acid payloads without killing them. Some typical microprojectiles consist of biologically inert substances such as tungsten platinum and gold beads. Methods of using particle bombardment are well known to those of skill in the art.

See for example Klein et al. Particle bombardment a universal approach for gene transfer to cells and tissues 4 5 583 590 1993 and references cited therein.

In certain embodiments a variety of methods incorporating calcium phosphate co precipitation can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . Methods of using calcium phosphate co precipitation are well known to those of skill in the art. See for example Uskokovi et al. Nanosized hydroxyapatite and other calcium phosphates chemistry of formation and application as drug and gene delivery agents . Biomed. Mater. Res. B Appl. Biomater 96 1 152 191 2011 and references cited therein. See also Colosimo et al. Transfer and expression of foreign genes in mammalian cells 29 2 314 8 320 322 2000 and references cited therein.

In certain embodiments microinjection and sonication methods can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . Methods of using microinjection and sonication are well known to those of skill in the art. See for example Rochlitz et al. Gene therapy of cancer 131 1 2 4 9 2001 and references cited therein. See also Donnelly et al. Microneedle based drug delivery systems microfabrication drug delivery and safety 17 4 187 207 2010 and references cited therein. See also Miller et al. Sonoporation mechanical DNA delivery by ultrasonic cavitation 27 1 6 115 34 2002 and references cited therein.

In certain embodiments liposomes and lipid formulations can be used to deliver into a host cell tissue or organism target nucleic acids encoding modified forms of PE and other polynucleotides as needed to allow expression of the same . Liposomes are vesicular structures characterized by a phospholipid bilayer membrane and an inner aqueous medium. Multilamellar liposomes have multiple lipid layers separated by aqueous medium. They form spontaneously when phospholipids are suspended in an excess of aqueous solution. An example of a commonly used commercially available lipid formulation is Lipofectamine Gibco BRL . Methods of using liposomes and lipid formulations to deliver nucleic acids to cells tissues and organisms are well known to those of skill in the art. See for example Xiong et al. Cationic liposomes as gene delivery system transfection efficiency and new application 66 3 158 64 2011 and references cited therein. See also Pichon et al. Chemical vectors for gene delivery uptake and intracellular trafficking 21 5 640 645 2010 and references cited therein. See also Pathak et al. Recent trends in non viral vector mediated gene delivery 4 11 1559 1572 2009 .

Expression of modified forms of PE including fusions conjugates and otherwise linked molecules may be expressed in host cells tissues and organisms using gene switch expression systems. Some examples without limitation of such gene expression systems and genetically engineered cells comprising gene switch expression systems which can be used to express polynucleotides and polypeptides of the present invention are described in the following publications each of which are hereby incorporated by reference herein 

For purposes of expressing polynucleotides and polypeptides under control of a gene switch mechanism the term gene switch refers to the combination of a response element associated with a promoter and a ligand dependent transcription factor based system which in the presence of one or more ligands modulates the expression of a gene into which the response element and promoter are incorporated. Stated otherwise a gene switch refers to a peptide protein or polypeptide complex that functions to a bind an activating ligand and b regulate the transcription of a gene of interest in a ligand dependent fashion.

In one embodiment the polynucleotide encoding a gene switch is a recombinant polynucleotide i.e. a polynucleotide that has been engineered by molecular biological manipulation to encode the gene switch. In another embodiment the recombinant polynucleotide is a synthetic polynucleotide.

As used herein with respect to gene switch regulation systems the term dimerizes with the ligand binding domain that binds an activating ligand refers to a selective protein protein interaction that is induced by the presence of activating ligand.

As used herein the term ligand binding domain that binds an activating ligand refers to an amino acid sequence that selectively binds an activating ligand. In the methods disclosed herein an activating ligand binds to a ligand binding domain e.g. an ecdysone receptor ligand binding domain that is part of a ligand dependent transcriptional activation complex that regulates the expression of a polynucleotide sequence that encodes a gene of interest. Hence the expression of the gene of interest is regulated in a ligand dependent fashion.

The term ecdysone receptor based with respect to a gene switch refers to a gene switch comprising at least a functional part of a naturally occurring or synthetic ecdysone receptor ligand binding domain and which regulates gene expression in response to a ligand that binds to the ecdysone receptor ligand binding domain.

As used herein selective binding of an activating ligand to a ligand binding domain in a gene switch means that the ligand has an EC50 of about 700 nanomolar nM 650 nM 600 nM 550 nM 500 nM 450 nM 400 nM 350 nM 300 nM 250 nM 225 nM 200 nM 175 nM 150 nM 125 nM 100 nM 95 nM 90 nM 85 nM 80 nM 75 nM 70 nM 65 nM 60 nM 55 nM 50 nM 45 nM 40 nM 35 nM 30 nM 25 nM 20 nM 15 nM 10 nM 9 nM 8 nM 7 nM 6 nM 5 nM 4 nM 3 nM 2 nM or 1 nM or less in a gene switch assay.

As used herein EC50 is the half maximal effective concentration which refers to the concentration of an activating ligand that induces a gene switch regulated change in expression of a polynucleotide encoding an gene of interest e.g. modified forms of PE including fusions conjugates et cetera that is halfway between the baseline level of expression and the maximum level of expression after a specified exposure time. Examples of cellular assays for measuring gene switch regulated gene expression are well known to those of skill in the art. See for example Karzenowski et al. 39 191 200 2005 .

In one embodiment the ligand binding domain that binds an activating ligand e.g. an ecdysone receptor ligand binding domain dimerizes with another ligand binding domain e.g. a retinoid X receptor ligand binding domain to form a protein protein complex.

In one embodiment the expression of the gene of interest is regulated by an activating ligand in an on off fashion that is independent of the concentration or dosage of an activating ligand. In another embodiment the expression of the gene of interest is regulated by an activating ligand in a concentration or dosage dependent fashion i.e. there is a dose response relationship between the concentration or dosage of an activating ligand and the level of gene expression of the gene of interest. See e.g. US Patent Publication No. 2009 0123441 see also WO 2009 048560 PCT USUS2008 011563 .

The term operably linked refers to the association of polynucleotide sequences on a single polynucleotide so that the function of one is affected by the other. For example a promoter is operably linked with a coding sequence when it is capable of affecting the expression of that coding sequence i.e. that the coding sequence is under the transcriptional control of the promoter . Coding sequences can be operably linked to regulatory sequences in sense or antisense orientation.

In one embodiment an activating ligand or a composition thereof is administered to a subject orally. In another embodiment an activating ligand or a composition thereof is administered to a subject parenterally. In another embodiment an activating ligand or a composition thereof is administered subcutaneously intramuscularly intravenously intraperitoneally transdermally or intratumorally.

In one embodiment the ligand binding domain in the gene switch is a Group H nuclear receptor ligand binding domain or a mutant thereof that binds an activating ligand. In another embodiment the Group H nuclear receptor ligand binding domain is selected from the group consisting of an ecdysone receptor ligand binding domain a ubiquitous receptor ligand binding domain an orphan receptor 1 ligand binding domain an NER 1 ligand binding domain a receptor interacting protein 15 ligand binding domain a liver X receptor 3 ligand binding domain a steroid hormone receptor like protein ligand binding domain a liver X receptor ligand binding domain a liver X receptor ligand binding domain a farnesoid X receptor ligand binding domain a receptor interacting protein 14 ligand binding domain and a farnesol receptor ligand binding domain ligand binding domain or a mutant thereof that binds an activating ligand.

In another embodiment the Group H nuclear receptor ligand binding domain is an ecdysone receptor ligand binding domain or a mutant thereof that binds an activating ligand. In another embodiment the ecdysone receptor ligand binding domain is selected from the group consisting of an Arthropod ecdysone receptor ligand binding domain a Lepidopteran ecdysone receptor ligand binding domain a Dipteran ecdysone receptor ligand binding domain an Orthopteran ecdysone receptor ligand binding domain a Homopteran ecdysone receptor ligand binding domain and a Hemipteran ecdysone receptor ligand binding domain a spruce budworm ecdysone receptor ligand binding domain a beetle ecdysone receptor ligand binding domain a ecdysone receptor ligand binding domain a ecdysone receptor ligand binding domain a midge ecdysone receptor ligand binding domain a silk moth ecdysone receptor ligand binding domain a squinting bush brown ecdysone receptor ligand binding domain a buckeye ecdysone receptor ligand binding domain a fruit fly ecdysone receptor ligand binding domain a mosquito ecdysone receptor ligand binding domain a blowfly ecdysone receptor ligand binding domain a blowfly ecdysone receptor ligand binding domain a blowfly ecdysone receptor ligand binding domain a Mediterranean fruit fly ecdysone receptor ligand binding domain a locust ecdysone receptor ligand binding domain an aphid ecdysone receptor ligand binding domain a fiddler crab ecdysone receptor ligand binding domain an ixodid tick ecdysone receptor ligand binding domain a whitefly Bamecia argentifoli ecdysone receptor ligand binding domain a leafhopper ecdysone receptor ligand binding domain or a mutant thereof that binds An activating ligand.

In another embodiment the ecdysone receptor ligand binding domain is a spruce budworm ecdysone receptor ligand binding domain for which the amino acid sequence is 

Exemplary ecdysone receptor ligand binding domains include those disclosed for example in U.S. Pat. No. 7 935 510 see also WO 2003 0 27289 PCT US2002 005026 U.S. Pat. No. 7 919 269 see also WO 2003 027266 PCT US 2002 05234 U.S. Pat. No. 7 563 879 see also WO 2003 0 27289 PCT US2002 005026 and in U.S. Patent Publication No. 2006 0100416 A1 see also WO 2002 066612 PCT US2002 005090 each of which is hereby incorporated by reference in its entirety.

In one embodiment the ecdysone receptor ligand binding domain is a mutant of an ecdysone receptor ligand binding domain that binds the activating compound. In another embodiment the ecdysone receptor ligand binding domain is a mutant of the spruce budworm ecdysone receptor ligand binding domain that binds the activating compound.

In one embodiment the gene switch comprises a ecdysone receptor ligand binding domain that is engineered to contain the mutations V107I and Y127E of the ecdysone receptor sequence as set forth in SEQ ID NO 1 of U.S. Patent Publication No. 2006 0100416 see also WO 2002 066612 PCT US2002 005090 . The term V107I means that the valine amino acid residue at position 107 a as set forth in SEQ ID NO 1 of U.S. Patent Publication No. 2006 0100416 is changed to isoleucine. The term Y127E means that the tyrosine amino acid residue at position 127 as set forth in SEQ ID NO 1 of U.S. Patent Publication No. 2006 0100416 is changed to glutamate.

Exemplary mutant ecdysone receptor ligand binding domains are disclosed for example in US 2006 0100416 A1 see also WO 2002 066612 PCT US2002 005090 and U.S. Pat. No. 7 935 510 Pub. No. 2005 0266457 see also WO 2005 108617 PCT US2005 015089 each of which is incorporated by reference in its entirety.

In one embodiment the gene switch comprises a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand. In one embodiment the ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand is a Group B nuclear receptor ligand binding domain. In another embodiment the Group B nuclear receptor ligand binding domain is selected from the group consisting of a retinoid X receptor ligand binding domain an H 2 region II binding protein ligand binding domain a nuclear receptor co regulator 1 ligand binding domain an ultraspiracle protein ligand binding domain a 2Cl nuclear receptor ligand binding domain and a chorion factor 1 ligand binding domain. In another embodiment a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand is not an ecdysone receptor ligand binding domain.

In one embodiment the ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand is a retinoic X receptor ligand binding domain. In another embodiment the retinoic X receptor ligand binding domain is a vertebrate retinoic X receptor ligand binding domain. In another embodiment the retinoic X receptor ligand binding domain is a retinoic X receptor ligand binding domain. In another embodiment the retinoic X receptor ligand binding domain is a retinoic X receptor c isoform. In another embodiment the retinoic X receptor ligand binding domain is a retinoic X receptor isoform. In another embodiment the retinoic X receptor ligand binding domain is a retinoic X receptor isoform.

In another embodiment the retinoic X receptor ligand binding domain is an invertebrate retinoic X receptor ligand binding domain. In another embodiment the invertebrate retinoic X receptor ligand binding domain is a retinoic X receptor ligand binding domain.

In another embodiment the invertebrate retinoic X receptor ligand binding domain is a non Lepidopteran non Dipteran retinoic X receptor ligand binding domain.

In one embodiment the retinoid receptor ligand binding domain is a vertebrate retinoid X receptor ligand binding domain an invertebrate retinoid X receptor ligand binding domain an ultraspiracle protein ligand binding domain or a chimeric retinoid X receptor ligand binding domain.

In one embodiment the chimeric retinoid X receptor ligand binding domain comprises two polypeptide fragments wherein the first polypeptide fragment is from a vertebrate retinoid X receptor ligand binding domain an invertebrate retinoid X receptor ligand binding domain or an ultraspiracle protein ligand binding domain and the second polypeptide fragment is from a different vertebrate retinoid X receptor ligand binding domain a different invertebrate retinoid X receptor ligand binding domain or a different ultraspiracle protein ligand binding domain.

In another embodiment the chimeric retinoid X receptor ligand binding domain is one that is disclosed in U.S. Pat. No. 7 531 326 which is hereby incorporated by reference in its entirety.

In another embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 6 helices 1 7 helices 1 8 helices 1 9 helices 1 10 helices 1 11 or helices 1 12 of a first species of retinoid X receptor and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 7 12 helices 8 12 helices 9 12 helices 10 12 helices 11 12 helix 12 or F domain of a second species of retinoid X receptor respectively.

In another embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 6 of a first species RXR according to the disclosure and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 7 12 of a second species of retinoid X receptor.

In another embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 7 of a first species retinoid X receptor according to the disclosure and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 8 12 of a second species retinoid X receptor.

In another embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 8 of a first species of retinoid X receptor and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 9 12 of a second species of retinoid X receptor.

In another embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 9 of a first species of retinoid X receptor and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 10 12 of a second species of retinoid X receptor.

In another embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 10 of a first species of retinoid X receptor and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 11 12 of a second species of retinoid X receptor.

In another embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 11 of a first species of retinoid X receptor and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helix 12 of a second species of retinoid X receptor.

In another preferred embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 12 of a first species of retinoid X receptor and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises an F domain of a second species of retinoid X receptor.

In one embodiment the first polypeptide fragment in the chimeric retinoid X receptor ligand binding domain is human retinoid X receptor sequence and the second polypeptide fragment in the chimeric retinoid X receptor ligand binding domain is invertebrate retinoid X receptor sequence. In another embodiment the invertebrate retinoid X receptor sequence is retinoid X receptor sequence.

In another embodiment the first polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 1 8 of a human retinoid X receptor and the second polypeptide fragment of the chimeric retinoid X receptor ligand binding domain comprises helices 9 12 of retinoid X receptor.

In one embodiment the gene switch further comprises a DNA binding domain DBD . In another embodiment the DBD is selected from the group consisting of a GAL4 DBD a LexA DBD a transcription factor DBD a steroid thyroid hormone nuclear receptor superfamily member DBD a bacterial LacZ DBD and a yeast DBD.

In one embodiment the gene switch further comprises a transactivation domain TD . In another embodiment the transactivation domain is selected from the group consisting of a VP16 TD a GAL4 TD an NF B TD a BP64 TD and a B42 acidic TD.

In one embodiment a DNA binding domain the ligand binding domain that binds an activating ligand a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand and a transactivation domain are encoded by polynucleotide sequences that are contained in the same polynucleotide.

In another embodiment a DNA binding domain a ligand binding domain that binds an activating ligand a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand and a transactivation domain are encoded by polynucleotide sequences that are contained in two or more separate polynucleotide sequences.

In another embodiment a DNA binding domain a ligand binding domain that binds an activating ligand a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand and a transactivation domain are encoded by polynucleotide sequences that are contained in two separate polynucleotide sequences.

In another embodiment a DNA binding domain and a ligand binding domain that binds an activating ligand are encoded by polynucleotide sequences that are contained in a first polynucleotide sequence and a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand and a transactivation domain are encoded by polynucleotide sequences that are contained in a second polynucleotide sequence.

In another embodiment a DNA binding domain and a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand are encoded by polynucleotide sequences that are contained in a first polynucleotide sequence and a ligand binding domain that binds an activating ligand and a transactivation domain are encoded by polynucleotide sequences that are contained in a second polynucleotide sequence.

In embodiments in which one or more of the DNA binding domain a ligand binding domain that binds an activating ligand a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand and a transactivation domain are encoded by polynucleotide sequences that are contained in one or more separate polynucleotide sequences then the one or more separate polynucleotide sequences are operably linked to one or more separate promoters. In another embodiment the one or more separate polynucleotide sequences are operably linked to one or more separate enhancer elements. In another embodiment the promoter s and or the enhancer s are constitutively active. In another embodiment the promoter s and or the enhancer s are tissue specific promoters and or enhancers.

In one embodiment the gene switch comprises a DNA binding domain an ecdysone receptor ligand binding domain a ligand binding domain that dimerizes with the ecdysone receptor ligand binding domain and a transactivation domain.

In another embodiment the gene switch comprises a DNA binding domain an ecdysone receptor ligand binding domain a retinoid X receptor ligand binding domain and a transactivation domain.

In another embodiment the gene switch comprises a DNA binding domain an ecdysone receptor ligand binding domain a chimeric vertebrate invertebrate retinoid X receptor ligand binding domain and a transactivation domain.

In another embodiment the gene switch comprises a GAL4 DNA binding domain a ecdysone receptor ligand binding domain that is engineered to contain the mutations V107I and Y127E of the ecdysone receptor sequence set forth in SEQ ID NO 1 a chimeric retinoid X receptor ligand binding and a VP16 transactivation domain.

In another embodiment the host cell further comprises a polynucleotide encoding a peptide protein or polypeptide whose expression is regulated by the gene switch. A promoter that binds the gene switch complex is operably linked to the polynucleotide encoding a peptide protein or polypeptide whose expression is regulated by the gene switch.

In another embodiment the polynucleotide encoding a peptide protein or polypeptide whose expression is regulated by the gene switch is contained in the same polynucleotide as a polynucleotide that encodes one or more of a DNA binding domain the ligand binding domain that binds an activating ligand a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand and a transactivation domain. Such constructs are disclosed for example in U.S. Patent Publication No. 2009 0123441 see also WO 2009 048560 PCT USUS2008 011563 .

In another embodiment the polynucleotide encoding a peptide protein or polypeptide whose expression is regulated by the gene switch is contained in a different nucleic acid molecule than a nucleic acid molecule that encodes one or more of a DNA binding domain the ligand binding domain that binds an activating ligand a ligand binding domain that dimerizes with the ligand binding domain that binds an activating ligand and a transactivation domain.

In one embodiment the gene switch is more sensitive to an activating ligand than to a steroid hormone. In another embodiment the gene switch is more sensitive to an activating ligand than to another diacylhydrazine compound.

The sensitivity of a gene switch to an activating ligand relative to another ligand can readily be determined in an in vitro assay for example an in vitro assay that employs a reporter gene such as firefly luciferase. Examples of such in vitro assays are well known to those of ordinary skill in the art. See for example Karzenowski et al. 39 191 200 2005 .

In one embodiment the polynucleotide encoding the gene switch is contained in a vector. In one embodiment the vector selected from the group consisting of a plasmid an expression vector a replicon a phage vector a cosmid a viral vector a liposome an electrically charged lipid e.g. a cytofectin a DNA protein complex and a biopolymer.

In another embodiment the vector is a retroviral vector. In another embodiment the vector is selected from the group consisting of an adeno associated viral vector a pox viral vector a baculoviral vector a vaccinia viral vector a herpes simplex viral vector an Epstein Barr viral vector an adenoviral vector a gemini viral vector and a caulimo viral vector.

In one embodiment a composition of the invention comprises one or more polynucleotides that encode two or more orthogonal gene switches. Two or more individually operable gene regulation systems are said to be orthogonal when a modulation of each of the given gene switches by its respective ligand results in a measurable change in the magnitude of expression of the gene that is regulated by that gene switch and b the change is statistically significantly different than the change in expression of all other gene switches that are in the host cell. In one embodiment regulation of each individually operable gene switch system effects a change in gene expression at least 2 fold 3 fold 4 fold 5 fold 10 fold 20 fold 50 fold 70 fold 100 fold 200 fold 300 fold 400 fold or 500 fold greater than all of the other operable gene switches in the host cell. Non limiting examples of orthogonal gene switch systems are set forth in U.S. Pat. No. 8 105 825 Publication No. US 2002 0110861 A1 .

As used herein an activating ligand is a compound that binds selectively to the ligand binding domain of a gene switch.

In one embodiment the activating ligand is administered to the subject within an hour of the time at which the priming dosage is administered to the subject. In another embodiment the activating ligand is administered to the subject within about 24 48 96 120 144 or 168 hours of the time at which the priming dosage is administered to the subject. In another embodiment the activating ligand is administered to the subject within about 1 2 3 4 or 5 weeks of the time at which the priming dosage is administered to the subject.

In one embodiment the activating ligand is administered to the subject within an hour of the time at which the first of the at least one boosting dosage is administered to the subject. In another embodiment the activating ligand is administered to the subject within about 24 48 96 120 144 or 168 hours of the time at which the first of the at least one boosting dosage is administered to the subject. In another embodiment the activating ligand is administered to the subject within about 1 2 3 4 or 5 weeks of the time at which the first of the at least one boosting dosage is administered to the subject.

In another embodiment a composition of the invention is contained within a container. In one embodiment the container is a vial. In another embodiment the container is a multiple use vial. In another embodiment the container displays an expiration date for the composition. In another embodiment the container contains instructions for using the composition.

In one embodiment a composition of the invention is a unit dosage composition. In one embodiment a unit dosage composition is a composition that is manufactured to supply a single dosage of the composition of the invention. In another embodiment the unit dosage composition is manufactured to provide more than one measured dosages of the composition of the invention.

The present application also provides an article of manufacture comprising more than one of the unit dosage compositions of the invention. In one embodiment the article of manufacture is a container. In another embodiment the article of manufacture is a box. In another embodiment the article of manufacture displays an expiration date for the unit dosage composition.

The present invention also provides a kit comprising more than one of the composition or unit dosage of the present invention. In one embodiment the kit displays an expiration date for the composition or unit dosage. In another embodiment the kit displays and or or contains instructions for using the composition or unit dosage. In another embodiment the kit also comprises an activating ligand that binds to the ligand binding domain of the gene switch encoded by the polynucleotide in the composition or unit dosage.

The present invention also provides a drug label for the composition or unit dosage of the present invention. In one embodiment the drug label displays an expiration date for the composition or unit dosage. In another embodiment the drug label displays instructions for using the composition or unit dosage. In another embodiment the drug label displays the approved indication s for the composition or unit dosage. In another embodiment the said label is in paper form. In another embodiment the drug label is in digital or computer readable form.

The term activating ligand as used herein refers to a compound that shows activity as an ecdysone receptor agonist i.e. a compound that is able to mimic 20 hydroxyecdysone biological activity and binds to a gene switch ligand binding domain. Activating ligands for use in the present invention include both ecdysteroids and non steroidal compounds e.g. tebufenozide and methoxyfenozide.

In one embodiment the activating ligand is an ecdysone receptor agonist disclosed in U.S. Pat. No. 8 076 517 Publication No. 2009 0163592 No. 2009 0298175 No. 2005 0228016 and in U.S. Pat. Nos. 6 258 603 7 375 093 7 456 315 7 304 161 and 7 304 162 each of which are hereby incorporated by reference herein.

In another embodiment the activating ligand is a compound having Formula I wherein A B and E are defined according to Table 3.

In another embodiment the activating ligand is a compound having Formula I selected from the group consisting of 

3 5 Dimethyl benzoic acid N tert butyl N 3 hydroxymethyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

3 5 Dimethyl benzoic acid N tert butyl N 3 tert butyl dimethyl silanyloxymethyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

7 N tert Butyl N 3 5 dimethyl benzoyl hydrazinocarbonyl 8 methyl 2 3 dihydro benzo 1 4 dioxine 2 carboxylic acid 

7 N tert Butyl N 3 5 dimethyl benzoyl hydrazinocarbonyl 8 methyl 2 3 dihydro benzo 1 4 dioxine 2 carboxylic acid methyl ester 

3 5 Dimethyl benzoic acid N tert butyl N 3 semicarbazidomethyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

Phenyl carbamic acid 7 N tert butyl N 3 5 dimethyl benzoyl hydrazinocarbonyl 8 methyl 2 3 dihydro benzo 1 4 dioxin 2 ylmethyl ester 

3 5 Dimethyl benzoic acid N 3 2 amino ethyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl N tert butyl hydrazide 

7 N tert Butyl N 3 5 dimethyl benzoyl hydrazinocarbonyl 8 methyl 2 3 dihydro benzo 1 4 dioxine 2 carboxylic acid pentafluorophenyl ester 

7 N tert Butyl N 3 5 dimethyl benzoyl hydrazinocarbonyl 8 methyl 2 3 dihydro benzo 1 4 dioxine 2 carboxylic acid methylamide 

3 5 Dimethyl benzoic acid N tert butyl N 3 formyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

Toluene 4 sulfonic acid 7 N tert butyl N 3 5 dimethyl benzoyl hydrazinocarbonyl 8 methyl 2 3 dihydro benzo 1 4 dioxin 2 ylmethyl ester 

3 5 Dimethyl benzoic acid N tert butyl N 3 hydroxyimino methyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

3 5 Dimethyl benzoic acid N tert butyl N 3 cyanomethyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

3 5 Dimethyl benzoic acid N tert butyl N 5 methyl 3 methylsulfanylmethyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

3 5 Dimethyl benzoic acid N tert butyl N 3 methanesulfonylmethyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

3 5 Dimethyl benzoic acid N tert butyl N 3 fluoromethyl 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

3 5 Dimethyl benzoic acid N 1 tert butyl 3 4 4 trimethyl pent 2 enyl N 3 methoxy 2 methyl benzoyl hydrazide 

In another embodiment the activating ligand is an enantiomerically enriched compound having Formula II 

In another embodiment the activating ligand is an enantiomerically enriched compound having Formula III 

In another embodiment the activating ligand is an enantiomerically enriched compound having Formula III wherein 

In another embodiment the activating ligand is a compound having Formula III selected from the group consisting of 

 R 3 5 Dimethyl benzoic acid N 1 tert butyl butyl N 5 methyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

 R 3 5 Dimethyl benzoic acid N 1 tert butyl butyl N 5 ethyl 2 3 dihydro benzo 1 4 dioxine 6 carbonyl hydrazide 

 R 3 5 Dimethoxy 4 methyl benzoic acid N 1 tert butyl butyl N 3 methoxy 2 methyl benzoyl hydrazide and

In another embodiment the activating ligand is a compound having Formula IV wherein Q is O Ris methyl and R R R R R R R and Rare defined according to Table 4.

In another embodiment the activating ligand is a compound having Formula V wherein Q is oxygen and R R R and Rare defined according to Table 5.

In another embodiment the activating ligand is a compound having Formula VI wherein n is 2 and R R R and Rare defined according to Table 6.

In another embodiment the activating ligand is a compound having Formula VIII selected from the group consisting of 

3 Chloro N 3 3 5 dimethyl phenyl 5 ethyl 5 methyl 1 2 4 oxadiazol 4 yl 2 2 3 3 tetrafluoro propionamide 

N 3 3 5 Dimethyl phenyl 1 oxa 2 4 diaza spiro 4.5 7 8 benzo dec 2 en 4 yl 3 methoxy 2 methyl benzamide 

3 Chloro N 5 5 dimethyl 3 3 trifluoromethyl phenyl 1 2 4 oxadiazol 4 yl 2 2 3 3 tetrafluoro propionamide 

3 Chloro N 3 3 5 dimethyl phenyl 1 oxa 2 4 diaza spiro 4.4 non 2 en 4 yl 2 2 3 3 tetrafluoro propionamide 

Benzo b thiophene 2 carboxylic acid 5 5 dimethyl 3 3 trifluoromethyl phenyl 1 2 4 oxadiazol 4 yl amide 

Benzo b thiophene 2 carboxylic acid 5 5 dimethyl 3 4 trifluoromethoxy phenyl 1 2 4 oxadiazol 4 yl amide 

Benzo b thiophene 2 carboxylic acid 3 3 5 dimethyl phenyl 1 oxa 2 4 diaza spiro 4 4 non 2 en 4 yl amide 

Benzo b thiophene 2 carboxylic acid 3 3 5 dimethyl phenyl 5 ethyl 5 methyl 1 2 4 oxadiazol 4 yl amide 

5 Ethyl 2 3 dihydro benzo 1 4 dioxine 6 carboxylic acid 5 5 dimethyl 3 phenyl 1 2 4 oxadiazol 4 yl amide and

5 Ethyl 2 3 dihydro benzo 1 4 dioxine 6 carboxylic acid 3 3 5 dimethyl phenyl 5 ethyl 5 methyl 1 2 4 oxadiazol 4 yl amide.

In another embodiment the activating ligand is a compound selected from the group consisting of 20 hydroxyecdysone 2 methyl ether 20 hydroxyecdysone 3 methyl ether 20 hydroxyecdysone 14 methyl ether 20 hydroxyecdysone 2 22 dimethyl ether 20 hydroxyecdysone 3 22 dimethyl ether 20 hydroxyecdysone 14 22 dimethyl ether 20 hydroxyecdysone 22 25 dimethyl ether 20 hydroxyecdysone 2 3 14 22 tetramethyl ether 20 hydroxyecdysone 22 n propyl ether 20 hydroxyecdysone 22 n butyl ether 20 hydroxyecdysone 22 allyl ether 20 hydroxyecdysone 22 benzyl ether 20 hydroxyecdysone 22 28R S 2 ethyloxiranyl ether ponasterone A 2 methyl ether ponasterone A 14 methyl ether ponasterone A 22 methyl ether ponasterone A 2 22 dimethyl ether ponasterone A 3 22 dimethyl ether ponasterone A 14 22 dimethyl ether dacryhainansterone 22 methyl ether 25 26 didehydroponasterone A iso stachysterone C 25 26 shidasterone stachysterone D stachysterone C 22 deoxy 20 hydroxyecdysone taxisterone ponasterone A polyporusterone B 22 dehydro 20 hydroxyecdysone 20 hydroxyecdysone pterosterone 25R inokosterone 25S inokosterone pinnatasterone 25 fluoroponasterone A 24 28 dehydromakisterone A 24 epi makisterone A makisterone A 20 hydroxyecdysone 22 methyl ether 20 hydroxyecdysone 25 methyl ether abutasterone 22 23 di epi geradiasterone 20 26 dihydroxyecdysone podecdysone C 24 epi abutasterone geradiasterone 29 norcyasterone ajugasterone B 24 28 Z dehydroamarasterone B amarasterone A makisterone C rapisterone C 20 hydroxyecdysone 22 25 dimethyl ether 20 hydroxyecdysone 22 ethyl ether carthamosterone 24 25 dehydroprecyasterone leuzeasterone cyasterone 20 hydroxyecdysone 22 allyl ether 24 28 Z dehydro 29 hydroxymakisterone C 20 hydroxyecdysone 22 acetate viticosterone E 20 hydroxyecdysone 25 acetate 20 hydroxyecdysone 22 n propyl ether 24 hydroxycyasterone ponasterone A 22 hemisuccinate 22 acetoacetyl 20 hydroxyecdysone canescensterone 20 hydroxyecdysone 22 hemisuccinate inokosterone 26 hemisuccinate 20 hydroxyecdysone 22 benzoate 20 hydroxyecdysone 22 D glucopyranoside 20 hydroxyecdysone 25 D glucopyranoside sileneoside A 20 hydroxyecdysone 22 galactoside 3 deoxy 1 20 dihydroxyecdysone 3 deoxyintegristerone A 2 deoxyintegristerone A 1 epi integristerone A integristerone A sileneoside C integristerone A 22 galactoside 2 22 dideoxy 20 hydroxyecdysone 2 deoxy 20 hydroxyecdysone 2 deoxy 20 hydroxyecdysone 3 acetate 2 deoxy 20 26 dihydroxyecdysone 2 deoxy 20 hydroxyecdysone 22 acetate 2 deoxy 20 hydroxyecdysone 3 22 diacetate 2 deoxy 20 hydroxyecdysone 22 benzoate ponasterone A 2 hemisuccinate 20 hydroxyecdysone 2 acetate 20 hydroxyecdysone 2 hemisuccinate 20 hydroxyecdysone 2 D glucopyranoside 2 dansyl 20 hydroxyecdysone 20 hydroxyecdysone 2 22 dimethyl ether ponasterone A 3 D xylopyranoside limnantheoside B 20 hydroxyecdysone 3 methyl ether 20 hydroxyecdysone 3 acetate 20 hydroxyecdysone 3 D xylopyranoside limnantheoside A 20 hydroxyecdysone 3 D glucopyranoside sileneoside D 20 hydroxyecdysone 3 galactoside 20 hydroxyecdysone 3 D glucopyranosyl 1 3 D xylopyranoside limnantheoside C cyasterone 3 acetate 2 dehydro 3 epi 20 hydroxyecdysone 3 epi 20 hydroxyecdysone coronatasterone rapisterone D 3 dehydro 20 hydroxyecdysone 5 hydroxy 25 26 didehydroponasterone A 5 hydroxystachysterone C 25 deoxypolypodine B polypodine B 25 fluoropolypodine B 5 hydroxyabutasterone 26 hydroxypolypodine B 29 norsengosterone sengosterone 6 hydroxy 20 hydroxyecdysone 6 hydroxy 20 hydroxyecdysone 20 hydroxyecdysone 6 oxime ponasterone A 6 carboxymethyloxime 20 hydroxyecdysone 6 carboxymethyloxime ajugasterone C rapisterone B muristerone A atrotosterone B atrotosterone A turkesterone 2 acetate punisterone rhapontisterone turkesterone atrotosterone C 25 hydroxyatrotosterone B 25 hydroxyatrotosterone A paxillosterone turkesterone 2 22 diacetate turkesterone 22 acetate turkesterone 11 acetate turkesterone 2 11 diacetate turkesterone 11 propionate turkesterone 11 butanoate turkesterone 11 hexanoate turkesterone 11 decanoate turkesterone 11 laurate turkesterone 11 myristate turkesterone 11 arachidate 22 dehydro 12 hydroxynorsengosterone 22 dehydro 12 hydroxycyasterone 22 dehydro 12 hydroxysengosterone 14 deoxy 14 H 20 hydroxyecdysone 20 hydroxyecdysone 14 methyl ether 14 perhydroxy 20 hydroxyecdysone 20 hydroxyecdysone 2 3 14 22 tetramethyl ether 20S 22 deoxy 20 21 dihydroxyecdysone 22 25 dideoxyecdysone 22S 20 2 2 dimethylfuranyl ecdysone 22R 20 2 2 dimethylfuranyl ecdysone 22 deoxyecdysone 25 deoxyecdysone 22 dehydroecdysone ecdysone 22 epi ecdysone 24 methylecdysone 20 deoxymakisterone A ecdysone 22 hemisuccinate 25 deoxyecdysone 22 D glucopyranoside ecdysone 22 myristate 22 dehydro 20 iso ecdysone 20 iso ecdysone 20 iso 22 epi ecdysone 2 deoxyecdysone sileneoside E 2 deoxyecdysone 3 glucoside blechnoside A 2 deoxyecdysone 22 acetate 2 deoxyecdysone 3 22 diacetate 2 deoxyecdysone 22 D glucopyranoside 2 deoxyecdysone 25 D glucopyranoside 2 deoxy 21 hydroxyecdysone 3 epi 22 iso ecdysone 3 dehydro 2 deoxyecdysone silenosterone 3 dehydroecdysone 3 dehydro 2 deoxyecdysone 22 acetate ecdysone 6 carboxymethyloxime ecdysone 2 3 acetonide 14 epi 20 hydroxyecdysone 2 3 acetonide 20 hydroxyecdysone 2 3 acetonide 20 hydroxyecdysone 20 22 acetonide 14 epi 20 hydroxyecdysone 2 3 20 22 diacetonide paxillosterone 20 22 p hydroxybenzylidene acetal poststerone 20R dihydropoststerone 20S dihydropoststerone poststerone 20 dansylhydrazine 20S dihydropoststerone 2 3 20 tribenzoate 20R dihydropoststerone 2 3 20 tribenzoate 20R dihydropoststerone 2 3 acetonide 20S dihydropoststerone 2 3 acetonide 5 H dihydrorubrosterone 2 14 22 25 tetradeoxy 5 ecdysone 5 ketodiol bombycosterol 2 3 22S 25 tetrahydroxy 5 cholestan 6 one 5 H 2 deoxy 21 hydroxyecdysone castasterone 24 epi castasterone 5 H 2 deoxyintegristerone A 5 H 22 deoxyintegristerone A 5 H 20 hydroxyecdysone 24 25 didehydrodacryhaninansterone 25 26 didehydrodacryhainansterone 5 deoxykaladasterone dacryhainansterone 14 H 14 deoxy 25 hydroxydacryhainansterone 25 hydroxydacryhainansterone rubrosterone 5 H dihydrorubrosterone dihydrorubrosterone 17 acetate sidisterone 20 hydroxyecdysone 2 3 22 triacetate 14 deoxy 14 H 20 hydroxyecdysone 14 epi 20 hydroxyecdysone 9 20 dihydroxyecdysone malacosterone 2 deoxypolypodine B 3 D glucopyranoside ajugalactone cheilanthone B 2 3 6 trihydroxy 5 cholestane 2 3 6 trihydroxy 5 cholestane 14 dehydroshidasterone stachysterone B 2 3 9 20R 22R 25 hexahydroxy 5 cholest 7 14 dien 6 one kaladasterone 14 H 14 deoxy 25 hydroxydacryhainansterone 4 dehydro 20 hydroxyecdysone 14 methyl 12 en shidasterone 14 methyl 12 en 15 20 dihydroxyecdysone podecdysone B 2 3 20R 22R tetrahydroxy 25 fluoro 5 cholest 8 14 dien 6 one 25 fluoropodecdysone B calonysterone 14 deoxy 14 18 cyclo 20 hydroxyecdysone 9 14 epoxy 20 hydroxyecdysone 9 14 epoxy 20 hydroxyecdysone 9 14 epoxy 20 hydroxyecdysone 2 3 20 22 diacetonide 28 homobrassinolide and iso homobrassinolide.

The disclosure of all patents patent applications and publications cited herein are incorporated by reference in their entireties.

The following examples are illustrative but not limiting of the methods of the present invention. Other suitable modifications and adaptations of the variety of conditions and parameters normally encountered in medical treatment and gene expression systems and which are obvious to those skilled in the art are within the spirit and scope of the invention.

In certain embodiments polynucleotides and polypeptides of the invention can be administered as part of a medicament or pharmaceutical composition. Medicaments and pharmaceutical compositions of the invention comprise one or more pharmaceutically acceptable carriers diluents excipients or additives.

The term excipient as used herein is typically an inert substance added to a composition to facilitate processing handling administration et cetera of a pharmaceutically acceptable composition. Useful excipients include but are not limited to adjuvants anti adherents binders carriers disintegrants fillers flavors colors diluents lubricants glidants preservatives sorbents solvents surfactants and sweeteners.

A few examples of pharmaceutically acceptable carriers diluents excipients and additives include without limitation water saline Ringer s solution dextrose solution buffers such as phosphates e.g. calcium phosphates such as tricalcium phosphate or calcium hydrogen phosphate citrate succinate acetic acid and other organic acids or their salts antioxidants proteins and other high molecular weight molecules such as serum albumin gelatin or immunoglobulins hydrophilic polymers such as polyvinylpyrrolidone amino acids such as glycine glutamic acid aspartic acid and arginine saccharides for example monosaccharides disaccharides lactose sucrose mannitol sorbitol other carbohydrates and sugar alcohols cellulose or its derivatives glucose mannose and dextrins chelating agents such as EDTA sugar alcohols such as mannitol or sorbitol counterions such as sodium surfactants such as polysorbates poloxamers or polyethylene glycol PEG and binders such as starch paste e.g. maize starch wheat starch rice starch potato starch gelatin tragacanth methyl cellulose hydroxypropylmethylcellulose sodium carboxymethylcellulose and or polyvinyl pyrrolidone .

Pharmaceutically acceptable carriers diluents excipients and additives may include disintegrating agents such as the above mentioned starches as well as compounds such as carboxymethyl starch cross linked polyvinyl pyrrolidone agar or alginic acid or a salt thereof such as sodium alginate and flow regulating agents and lubricants for example silica talc stearic acid or salts thereof such as magnesium stearate or calcium stearate and or polyethylene glycol. In one embodiment dragee cores are provided with suitable coatings which if desired are resistant to gastric juices. For this purpose concentrated saccharide solutions may be used which may optionally contain gum arabic talc polyvinyl pyrrolidone polyethylene glycol and or titanium dioxide lacquer solutions and suitable organic solvents or solvent mixtures. In order to produce coatings resistant to gastric juices solutions of suitable cellulose preparations such as acetylcellulose phthalate or hydroxypropylmethyl cellulose phthalate are used. Dye stuffs or pigments may be added to the tablets or dragee coatings for example for identification or in order to characterize combinations of active compound doses.

Pharmaceutical preparations which can be used orally include push fit capsules made of gelatin as well as soft sealed capsules made of gelatin and a plasticizer such as glycerol or sorbitol. The push fit capsules can contain the active compounds in the form of granules or nanoparticles which may optionally be mixed with fillers such as lactose binders such as starches and or lubricants such as talc or magnesium stearate and optionally stabilizers. In one embodiment the is dissolved or suspended in suitable liquids such as fatty oils or liquid paraffin optionally with stabilizers.

Fatty oils may comprise mono di or triglycerides. Mono di and triglycerides include those that are derived from C6 C8 C10 C12 C14 C16 C18 C20 and C22 acids. Exemplary diglycerides include in particular diolein dipalmitolein and mixed caprylin caprin diglycerides. Preferred triglycerides include vegetable oils fish oils animal fats hydrogenated vegetable oils partially hydrogenated vegetable oils synthetic triglycerides modified triglycerides fractionated triglycerides medium and long chain triglycerides structured triglycerides and mixtures thereof. Exemplary triglycerides include almond oil babassu oil borage oil blackcurrant seed oil canola oil castor oil coconut oil corn oil cottonseed oil evening primrose oil grapeseed oil groundnut oil mustard seed oil olive oil palm oil palm kernel oil peanut oil rapeseed oil safflower oil sesame oil shark liver oil soybean oil sunflower oil hydrogenated castor oil hydrogenated coconut oil hydrogenated palm oil hydrogenated soybean oil hydrogenated vegetable oil hydrogenated cottonseed and castor oil partially hydrogenated soybean oil partially soy and cottonseed oil glyceryl tricaproate glyceryl tricaprylate glyceryl tricaprate glyceryl triundecanoate glyceryl trilaurate glyceryl trioleate glyceryl trilinoleate glyceryl trilinolenate glyceryl tricaprylate caprate glyceryl tricaprylate caprate laurate glyceryl tricaprylate caprate linoleate and glyceryl tricaprylate caprate stearate.

In one embodiment the triglyceride is the medium chain triglyceride available under the trade name LABRAFAC CC. Other triglycerides include neutral oils e.g. neutral plant oils in particular fractionated coconut oils such as known and commercially available under the trade name MIGLYOL including the products MIGLYOL 810 MIGLYOL 812 MIGLYOL 818 and CAPTEX 355. Other triglycerides are caprylic capric acid triglycerides such as known and commercially available under the trade name MYRITOL including the product MYRITOL 813. Further triglycerides of this class are CAPMUL MCT CAPTEX 200 CAPTEX 300 CAPTEX 800 NEOBEE M5 and MAZOL 1400.

Pharmaceutical compositions comprising triglycerides may further comprise lipophilic and or hydrophilic surfactants which may form clear solutions upon dissolution with an aqueous solvent. One such surfactant is tocopheryl polyethylene glycol 1000 succinate vitamin E TPGS . Examples of such compositions are described in U.S. Pat. No. 6 267 985.

In another embodiment the pharmaceutically acceptable carrier comprises LABRASOL Gattefosse SA which is PEG 8 caprylic capric glycerides. In another embodiment the pharmaceutically acceptable carrier comprises PL90G vitamin E TPGS and Miglyol 812N.

Pharmaceutical compositions can be administered in any suitable manner as determined by those skilled in the art such as but without limitation by oral rectal vaginal topical including dermal buccal and sublingual parenteral intravenous intraperitoneal intramuscular intratumoral intraarticular subcutaneous intranasal inhalation intradermal intrathecal epidural and by naso gastric routes.

Methods and compositions for preparation formulation and delivery of pharmaceutically acceptable compositions and medicaments are well known and routinely practiced by those skilled in the art. A few examples of textbooks and manuals providing information and instruction on such methods and compositions include Rowe et al. Editor Pharmaceutical Press 6Ed. August 2009 University of the Sciences in Philadelphia Editor Lippincott Williams Wilkins 21Ed. 2005 2011 PDR Network 2010 2012 PDR Network 2011 O Neil 14Ed. 2006 Allen et al. Editor Lippincott Williams Wilkins 9Ed. 2011 and Ash et al. Editor Synapse Information Resources Inc. 3Ed. 2007 .

Protocols for general molecular biology methods can be found in series editor J M Walker Humana Press New York.

Embodiments of the invention comprise any amino acid substituted form of PE as indicated by or represented in Table 13. Embodiments of the invention further comprise any amino acid substituted form of PE which may comprise any combination of amino acid substitutions indicated by or represented in Table 13.

Embodiments of the invention also comprise variants derivatives or biologically active fragments of any amino acid substituted form of PE as indicated by or represented in Table 13 wherein said variant derivative or biologically active fragment of PE is at least 80 identical at least 85 identical at least 90 identical at least 95 identical at least 97 identical at least 98 identical at least 99 identical or is at least 100 identical to an amino acid substituted form of PE or a fragment thereof as indicated by or represented in Table 13. For example embodiments of the invention comprise variants derivatives or biologically active fragments of any amino acid substituted form of PE as indicated by or represented in Table 13 wherein said variant derivative or biologically active fragment of PE is at least 80 identical at least 85 identical at least 90 identical at least 95 identical at least 97 identical at least 98 identical at least 99 identical or is at least 100 identical to PE constructs or fragments thereof as represented by pIEX02 003 through pIEX02 248 in Table 13 such as for example as shown in SEQ ID NO 177 pIEX02 228 SEQ ID NO 178 pIEX02 244 and SEQ ID NO 179 pIEX02 246 .

Embodiments of the invention include methods of making methods of using methods of treatment using medicaments comprising pharmaceutically acceptable compositions comprising therapeutically useful compositions comprising and kits comprising any of the amino acid substituted forms of PE referenced or otherwise described or provided for herein.

Peptides spanning the sequence of an approximately 38 kD form of exotoxin A protein PE38 were analyzed for the presence of immunogenic CD4 T cell epitopes using EPISCREEN T cell epitope mapping analysis Antitope Ltd Cambridge UK .

EPISCREEN is a proprietary technology commercially available through Antitope Ltd Cambridge UK to map T cell epitopes within a protein sequence to determine potential for immunogenicity based on the number and potency of T cell epitopes within a sequence . EPISCREEN T cell epitope mapping typically uses CD8 T cell depleted PBMCs from a minimum of 50 HLA typed donors selected to represent the human population of interest . Typically 15mer peptides with 12 amino acid overlaps spanning a protein sequence are analyzed in a large number of replicate cultures for in vitro CD4 T cell stimulation by 3H TdR incorporation. CD4 T cell stimulation is often detected in two or three adjacent and overlapping peptides since the core 9mer that binds the MHC class II binding groove will be present in more than one peptide sequence. Following identification of peptides that stimulate CD4 T cells in vitro in silico technology can be used to design epitope depleted deimmunized variants by determining the precise location of core 9mer sequences and the location of key MHC class II anchor residues.

A total of 120 overlapping 15mer peptides spanning the entire PE38 sequence SEQ ID NO 2 including 4 peptides covering a null mutation and 4 peptides spanning an N terminal linker sequence SEQ ID NO 3 were tested against a cohort of 52 healthy donors. CD4 T cell responses against individual peptides were measured using proliferation assays 3H thymidine incorporation . The proliferation assay data was used to compile a T cell epitope map of the PE38 sequence and six T cell epitopes were identified.

Peripheral blood mononuclear cells PBMC were isolated from healthy community donor buffy coats from blood drawn within 24 hours obtained from the UK National Blood Transfusion Service Addenbrooke s Hospital Cambridge UK and according to approval granted by Addenbrooke s Hospital Local Research Ethics Committee. PBMC were isolated from buffy coats by LYMPHOPREP Axis Shield UK Dundee Scotland density centrifugation. LYMPHOPREP is a ready made sterile and endotoxin tested solution for the isolation of human mononuclear cells from blood. See Axis Shield package insert for LYMPHOPREP density gradient media No. 619. March 03. Div. 1114740. CD8 T cells were depleted using CD8 ROSETTESEP STEMCELL Technologies Inc Manchester UK to remove CD8 cells from the isolated mononuclear cells. See e.g. StemCell Technologies Inc. ROSETTESEP procedure for Human CD8 T Cell Enrichment Cocktail Catalog 15023 15063 Procedure version 1.3.0 28572 May 2011 .

HLA allotypes of donors were characterized using the Biotest HLA SSP PCR tissue typing kit Biotest Solihull UK catalogue number 826215 . T cell responses to a reproducibility control neo antigen were also determined using Imject maricutlure keyhole limpet haemocyanin KLH Pierce Perbio Science UK Ltd Cramlington UK catalogue number 77600 with the KLH diluted to a final concentration of 100 g ml. PBMC were then frozen and stored in liquid nitrogen until required.

A cohort of 52 donors was selected to best represent the number and frequency of HLA DR allotypes expressed in the world population. Analysis of the allotypes expressed in the cohort against those expressed in the world population revealed that coverage of 80 was achieved and that all major HLA DR alleles individual allotypes with a frequency 5 expressed in the world population were well represented. Details of individual donor haplotypes and a comparison of the frequency of MHC class II haplotypes expressed in the world population and the sample population are shown in Table 7 and respectively.

Table 7. Donor details and haplotypes. Donor responses SI to KLH are shown for two independent proliferation assays. Test 1 was performed using KLH on freshly isolated PBMC and IEX01 is the KLH re test performed in the current study on PBMC recovered from liquid nitrogen storage as indicated above. Responses that did not produce the same result i.e. positive including borderline SI 1.90 p

PBMC from each donor were thawed counted and viability was assessed. Cells were revived in room temperature AIM V culture medium Invitrogen Paisley UK before adjusting the cell density to 2 3 10PBMC ml proliferation cell stock . Peptides were synthesized on a 1 3 mg scale with free N terminal amine and C terminal carboxylic acid. Peptides were dissolved in DMSO to a concentration of 10 mM and peptide culture stocks prepared by diluting into AIM V culture medium to a final concentration of 5 M in the well. For each peptide and each donor sextuplicate cultures were established in a flat bottomed 96 well plate. Both positive and negative control cultures were also tested in sextuplicate. For each donor three control antigen peptides KLH protein and peptides derived from Influenza A and Epstein Barr viruses were also included.

Cultures were incubated for a total of 6 days before adding 0.75 Ci 3 H thymidine PERKIN ELMER Beaconsfield UK to each well. Cultures were incubated for a further 18 hours before harvesting onto filter mats using a TOMTEC MACH III cell harvester TOMTEC Hamden Conn. USA . Counts per minute cpm for each well were determined by Meltilex PERKIN ELMER scintillation counting on a Microplate Beta Counter PERKIN ELMER in paralux low background counting mode.

For proliferation assays an empirical threshold of a stimulation index SI equal to or greater than 2 SI 2.00 has been previously established whereby samples inducing proliferative responses above this threshold are deemed positive where included borderline SI 1.90 are highlighted . Extensive assay development and previous studies have shown that this is the minimum signal to noise threshold allowing maximum sensitivity without detecting large numbers of false positive responses. Positive responses are defined by the following statistical and empirical thresholds 

In addition intra assay variation was assessed by calculating the coefficient of variance and standard deviation SD of the raw data from replicate cultures.

Proliferation assays were set up in sextuplicate cultures non adjusted data . To ensure that intra assay variability was low the data was also analyzed after removing the maximum and minimum cpm values adjusted data and the SI of donor responses was compared using both data sets.

T cell epitopes were identified by calculating the average frequency of positive responses defined above to all peptides in the study plus standard deviation SD to give a background response threshold. Any peptide that induced a frequency of positive proliferation responses above this threshold in both the adjusted and non adjusted data was considered to contain an immunogenic T cell epitope and thus potentially represents an immunogenicity inducing epitope which could give rise to immunogenic responses in vivo .

The sequences of peptides that were positive in the proliferation assay were analyzed using Antitope s predictive iTOPE software Perry et al. 2008 . This software predicts favorable interactions between amino acid side chains of the peptide and specific binding pockets within the MHC class II binding groove. Analysis of the peptide sequences using iTOPE was performed with overlapping 9mers spanning the peptides which were tested against each of the 34 MHC class II alleles. Each 9mer was scored based on the potential fit and interactions with the MHC class II molecules. 9mers that produced a high mean binding score were identified and from the T cell proliferation data 9mers which were considered as critical to T cell responses core 9mers were highlighted. iTOPE analysis was then repeated with a range of amino acid changes in the core 9mers in order to determine preferred amino acid substitutions for use in deimmunization.

A total of 120 peptides were synthesized spanning the entire PE38 sequence. The peptides were designed as 15mers to span the sequence in overlapping increments of 12 amino acids. These peptides were then tested for the presence of CD4 T cell epitopes by EPISCREEN T cell epitope mapping analysis. Positive T cell responses were defined by donors that produced a significant p

The output from non adjusted and adjusted data analysis was examined to ensure that intra assay variability was low and that positive responses were not the result of spurious proliferation in individual wells. The results from each analysis showed in most cases only small differences between the methods and donor responses for both non adjusted and adjusted analysis. Table 10 provides a summary of individual donor responses to each of the peptides. The proliferation assay data showing the frequency of positive donor responses to each peptide is shown in . For all peptides that induced a high frequency of positive SI 2.00 p

Peptides 50 52 and 53 induced a high number of positive T cell proliferation responses in the study cohort Table 8 and . Peptide 50 showed the highest number of positive responses with 15.38 donors responding in the non adjusted dataset and 15.38 in the adjusted data set SI 2.00 p

The magnitude of T cell proliferation responses can provide an indication as to the T cell precursor frequency. In general peptides that induce high frequency of positive responses in the study cohort and high magnitude T cell proliferation responses are a characteristic of recall like T cell responses in which the T cell pre cursor frequency is high. In contrast naive T cell responses are generally characterized by low magnitude T cell proliferation responses with low T cell precursor frequencies . Peptides 52 and 53 induced moderately high magnitude T cell proliferation responses where the mean SI for positive SI 2.00 p

A cluster of T cell responses were observed around peptides 65 68 and the subsequent analysis revealed the presence of two T cell epitopes in this region. Peptide 65 stimulated positive T cell proliferation responses in 15.38 of the study cohort for both non adjusted and adjusted datasets Table 8 and SI 2.00 p

Peptides 81 and 82 stimulated a number of T cell responses in the study cohort Table 8 and . Peptide 81 had the highest frequency of response of all the peptides tested with a frequency of positive responses of 19.23 in the non adjusted and 17.31 in the adjusted data set. For peptide 82 the frequency of positive response was 17.31 and 15.38 in the non adjusted and adjusted data sets respectively. The positive responses were relatively low in magnitude mean SI of positive responses ranged from 2.12 to 2.22 in the non adjusted and adjusted data sets suggesting that the T cell precursor frequency in healthy donors against this epitope is relatively low Table 9 . Adjacent peptide 80 induced a sub threshold response. In silico analysis of peptides 81 and 82 suggested a core 9mer of LRVYVPRSS SEQ ID NO 9 .

Peptide 110 induced positive T cell responses in 13.46 of the study cohort in non adjusted and 13.46 in adjusted datasets Table 8 and . The magnitude of positive proliferation responses was low with a mean SI of 2.23 for the non adjusted dataset and 2.07 for the adjusted dataset Table 9 . There was also a sub threshold response to peptide 111. In silico analysis of the peptides sequence revealed a core 9mer IPDKEQAIS SEQ ID NO 10 which in addition to peptide 110 was also present in peptide 111.

Table 9. Summary of magnitude mean SI and standard deviation and frequency donor response of positive T cell proliferation responses against peptides containing T cell epitopes for PE38. The position of p1 in potential core 9mers are shown as underlined bolded text as predicted by iTOPE in peptides 50 65 81 82 and 110.

Analysis of the responding donor haplotypes was performed whereby an association between MHC class II allotype and a response to a particular peptide was considered possible if the frequency of the allotype within the responding population was double the frequency observed in the study cohort. This analysis was only carried out for peptides that induced positive responses above the background response rate in the adjusted data in the study cohort and was also restricted to allotypes expressed at higher frequencies 5 in the study population.

Analysis of responding donor allotypes Table 10 and revealed that there was a possible association between T cell responses to peptides 81 and 82 and MHC class II allotype HLA DRB 1 07 which was expressed at twice the percentage of positively responding donors compared to the study population. Peptide 53 also had a possible association with DRB 1 11 and peptides 82 and 110 showed possible associations with DRB1 15. It should be noted that further studies such as MHC class II binding analysis would be required to show conclusively that responses to the T cell epitope are associated with these allotypes as the present analysis was performed on a small group of responding donors.

Table 10. Frequency expressed as a percentage of responding donor allotypes compared to the frequency of allotypes expressed in the IEX01 study cohort. An association between MHC class II allotype and a response to a particular epitope was considered if the frequency of the allotype within the responding population was double the frequency observed in the study population in the adjusted data set. Possible associations are indicated in heavily bordered boxes. The analysis has been restricted to allotypes expressed at higher frequencies 5 in the study population.

The results show that six T cell epitopes were present in the PE38 sequence. Table 6 Table 11 and summarize the location of the putative core 9mers in each sequence along with the frequency and magnitude of T cell responses against each epitope. The T cell epitopes identified in PE38 were prioritized according to their potency based on the frequency and magnitude mean SI of positive donor responses to each peptide. However since the responding donor magnitudes were similar Table 4 Table 9 for most epitopes the ranking was mainly based on frequency of positive donor responses from highest to lowest 

The six epitope core 9mer sequences were analyzed by proprietary software iTOPE in order to identify mutations that remove the T cell epitopes by eliminating or significantly reducing binding to MHC class II Table 11 . As part of the strategy as to which residues to mutate location within the structure was considered especially whether the residue is buried on the surface or near active sites.

Table 11. Projected mutations to remove MHC class II binding based upon iTOPE and crystal structure data .

EPISCREEN T cell epitope mapping of 120 overlapping 15mer peptides including 112 spanning the entire PE38 sequence suggested six novel T cell epitopes. In silico analysis was used to identify potential core 9mers for MHC binding and together with structural analysis was used as a basis for design of changes for re engineering and deimmunizing PE38 in particular and PE molecules in general.

The immunogenicity of amino acid substituted forms of PE can be assessed using the same procedures as described in Example 1. Accordingly EPISCREEN T cell epitope mapping analysis Antitope Ltd Cambridge UK analysis permits identification of amino acid substituted epitopes in PE polypeptides wherein the introduced amino acid changes result in reduced or undetectable immunogenicity i.e. for generating deimmunized forms of PE as compared to epitopes in corresponding forms of non amino acid substituted PE polypeptides.

EPISCREEN is a proprietary technology commercially available through Antitope Ltd Cambridge UK to map T cell epitopes within a protein sequence to determine potential for immunogenicity based on the number and potency of T cell epitopes within a sequence . EPISCREEN T cell epitope mapping typically uses CD8 T cell depleted PBMCs from a minimum of 50 HLA typed donors selected to represent the human population of interest . Typically 15mer peptides with 12 amino acid overlaps spanning a protein sequence are analyzed in a large number of replicate cultures for in vitro CD4 T cell stimulation by 3H TdR incorporation. CD4 T cell stimulation is often detected in two or three adjacent and overlapping peptides since the core 9mer that binds the MHC class II binding groove will be present in more than one peptide sequence. Following identification of peptides that stimulate CD4 T cells in vitro in silico technology can be used to design epitope depleted deimmunized variants by determining the precise location of core 9mer sequences and the location of key MHC class II anchor residues.

In this case amino acid substituted PE peptides are analyzed for the presence of immunogenic CD4 T cell epitopes using EPISCREEN T cell epitope mapping analysis. For example amino acid substituted 15mer peptides compared to non substituted 15mer peptides corresponding to a non amino acid substituted form of PE are tested against a cohort of healthy donors. CD4 T cell responses against individual peptides are measured using proliferation assays 3H thymidine incorporation . Proliferation assay data is used to compile a T cell epitope map of varying responses to amino acid substituted forms of PE to determine those amino acid changes producing reduced or abrogated immunogenic responses.

Peripheral blood mononuclear cells PBMC are isolated from healthy donor buffy coats e.g. from blood drawn within 24 hours . For example PBMC are isolated from buffy coats using density gradient centrifugation using LYMPHOPREP Axis Shield UK Dundee Scotland or a similar density gradient centrifugation media for the isolation of human mononuclear cells from blood such methods media and products are well known and routinely used by those skilled in the art . See e.g. Axis Shield package insert for LYMPHOPREP density gradient media No. 619. March 03. Div. 1114740. To remove CD8 cells from the isolated mononuclear cells CD8 T cells are depleted using CD8 ROSETTESEP kit STEMCELL Technologies Inc Manchester UK or similar CD8 selection methods and techniques such methods media and products are well known and routinely used by those skilled in the art . See e.g. StemCell Technologies Inc. ROSETTESEP procedure for Human CD8 T Cell Enrichment Cocktail Catalog 15023 15063 Procedure version 1.3.0 28572 May 2011 .

Donors HLA DR haplotypes are determined using methods or kits well known and routinely used by those skilled in the art. For example Donors HLA DR haplotypes are determined using a Biotest HLA SSP PCR tissue typing kit Biotest Solihull UK catalogue number 826215 . T cell responses to a reproducibility control antigen are measured using for example neo antigen using Imject maricutlure keyhole limpet haemocyanin KLH Pierce Perbio Science UK Ltd Cramlington UK catalogue number 77600 or other similar control antigen such antigens and methods are well known and routinely used by those skilled in the art . PBMC are frozen and stored in liquid nitrogen until ready for use in to measuring immunogenicity of amino acid substituted forms of PE.

A cohort of donors are selected to best represent the number and frequency of HLA DR allotypes expressed in the world population. It is desirable that allotypes expressed in the cohort represent a coverage of 80 of all major HLA DR alleles in the world population i.e. individual allotypes with a frequency 5 expressed in the world population are well represented . Records of individual donor haplotypes and comparison of the frequency of MHC class II haplotypes expressed in the world population and the sample population are recorded and assessed.

Donor responses SI to a control antigen such as KLH are assessed by comparing two independent proliferation assays. Test 1 is performed using the control antigen such as KLH on freshly isolated PBMC and Test 2 is the control antigen re test performed on PBMC recovered from liquid nitrogen storage the latter of which are used in assessing immunogenicity of amino acid substituted epitopes in PE. Responses that do not produce the same result in these two tests i.e. positive including borderline SI 1.90 p

PBMC from each donor are thawed counted and viability is assessed. Cells are revived in room temperature AIM V Culture Medium INVITROGEN Paisley UK before adjusting cell density to 2 3 10PBMC ml proliferation cell stock . Peptides are synthesized on a 1 3 mg scale with free N terminal amine and C terminal carboxylic acid. Peptides are dissolved in DMSO to a concentration of 10 mM and peptide culture stocks are prepared by diluting into AIM V Culture Medium to a final concentration of 5 M per well. For each peptide and each donor sextuplicate cultures are established in a flat bottomed 96 well plate. Both positive and negative control cultures are tested in sextuplicate. For each donor three control antigen peptides KLH protein and peptides derived from Influenza A and Epstein Barr viruses are also included.

Cultures are incubated for 6 days before adding 0.75 Ci 3 H thymidine PERKIN ELMER Beaconsfield UK to each well. Cultures are incubated a further 18 hours before harvesting onto filter mats using a TOMTEC MACH III cell harvester TOMTEC Hamden Conn. USA . Counts per minute cpm for each well are determined by MELTILEX PERKIN ELMER scintillation counting on a Microplate Beta Counter PERKIN ELMER in paralux low background counting mode.

In proliferation assays an empirical threshold of stimulation index SI equal to or greater than 2 SI 2.00 is considered to represent an induced proliferative response samples registering values above this threshold are deemed positive values of SI

In addition intra assay variation is assessed by calculating the coefficient of variance and standard deviation SD of raw data from replicate cultures.

Proliferation assays are set up in sextuplicate cultures from which non adjusted data is gathered. To ensure intra assay variability is low data is also analyzed after removing maximum and minimum cpm values to produce adjusted data and the SI of donor responses is compared using both data sets.

Reactive T cell epitopes are identified by calculating the average frequency of positive responses defined above to all peptides in the study plus standard deviation SD to give a background response threshold. Any peptide inducing a frequency of positive proliferation responses above the threshold in both adjusted and non adjusted data is considered to contain an immunogenic T cell epitope and thus potentially represents an immunogenicity inducing epitope which could give rise to immunogenic responses in vivo . Output from non adjusted and adjusted data is examined to ensure that intra assay variability is low and that positive responses are not the result of spurious proliferation in individual wells. An example of this type of analysis is provided in Example 1.

A comparison of corresponding forms of non amino acid substituted PE immunogenic epitope responses versus responses obtained with amino acid substituted PE peptides is used to assess and predict the effects of various amino acid substitutions in reducing or eliminating the immunogenicity of PE polypeptides i.e. for making deimmunized forms of PE .

Assays for measuring and testing the immunogenicity of amino acid substituted forms of PE may also be done as described and exemplified in Example 1 i.e. via proliferation assays quantitating CD4 T cell responses wherein amino acid substituted forms of PE i.e. deimmunized PE or DI PE and or DI PE conjugates and fusion proteins e.g. fusions of DI PE to antibodies or antigen binding fragments thereof are tested and measured for the presence and potency of immunogenic responses compared to responses induced by corresponding forms of non amino acid substituted PE peptides polypeptides and fusion or conjugation constructs.

Assays for measuring immunogenicity of amino acid substituted forms of PE specifically as indicated above or PE molecules generally may also be done according to methods routinely used and well known to those of skill in the art. For example immunogenicity of amino acid substituted forms of PE in particular or PE molecules in general as indicated above may be measured in vivo in non human primates and or in transgenic mouse model systems.

Assays for measuring the biological activity of amino acid substituted deimmunized forms of PE may be done according to methods routinely used and well known to those of skill in the art. Measured biological activities of deimmunized DI forms of PE DI PE in particular or PE molecules in general may include for example assays to measure 

In one example measurement of inhibition of protein synthesis may be done via use of in vitro transcription translation assays which are routinely used and well known to those of skill in the art . For example a cell free assay may be used to measure DI PE induced inhibition of in vitro transcription translation of a target plasmid such as but not limited to T7 luc . In the case of using a T7 luc transcription translation assay the biological activity readout would be chemiluminescent measurement of luciferase activity wherein amino acid substituted forms of PE are compared to corresponding non amino acid substituted forms of PE for ability inability to inhibit translation of the luciferase enzyme in vitro. In such assays the PE polypeptides being assayed can be introduced via expression from template DNA e.g. a PCR product encoding the toxin conjugate gene or by directly introducing quantified amounts of PE proteins. Such assays may be used to assess IC50 values of the various forms of PE tested IC50 concentration at which 50 of protein synthesis is inhibited versus standardized control samples lacking PE .

Some examples of kits and reagents available for in vitro transcription translation assays include but are not limited to 

Full protocols for use of such kits are provided by the manufacturer with each kit. A brief example of a typical experimental procedure may include 

Comparative protein synthesis inhibition values may show that various forms of DI PE exhibit 100 or about 100 of biological activity inhibition of protein synthesis compared to corresponding forms of non amino acid substituted PE. Comparative protein synthesis inhibition values may also show that various forms of DI PE exhibit at least 95 or at least about 95 at least 90 at least about 90 at least 85 at least about 85 at least 80 at least about 80 at least 75 at least about 75 at least 70 at least about 70 at least 60 at least about 60 at least 50 or at least about 50 of biological activity compared to corresponding forms of non amino acid substituted PE.

In one example measurement of cell cytotoxic activity may be done via use of in vitro cell based assays wherein deimmunized PE DI PE antibody conjugates are assayed in comparison to non amino acid substituted PE antibody conjugates. The antibody portion of such conjugates would be antibodies or antigen binding fragments thereof which specifically bind antigens expressed on the cell surface of cell types used in such in vitro assays. Cell cytotoxicity may be quantitated for example by measuring cell lysis wherein the biological readout is represented by measurement of for example based on chemiluminescent LUMI fluorometric FL and colorimetric COL outputs such as can be practiced using commercially available kits routinely used and well known to those of skill in the art.

Some examples of kits available for measurement and comparison of DI PE versus non amino acid substituted PE cell cytotoxicity include without limitation 

Full protocols for use of such kits are provided by the manufacturer with each kit. A brief example of a typical experimental procedure may include 

Comparative cell cytotoxicity values may show that various forms of DI PE exhibit 100 or about 100 of biological activity induction of cell cytotoxicity compared to corresponding forms of non amino acid substituted PE. Comparative cell cytotoxicity values may also show that various forms of DI PE exhibit at least 95 or at least about 95 at least 90 at least about 90 at least 85 at least about 85 at least 80 at least about 80 at least 75 at least about 75 at least 70 at least about 70 at least 60 at least about 60 at least 50 or at least about 50 of biological activity compared to corresponding forms of non amino acid substituted PE.

Quantitative in vitro transcription translation IVTT assays to assess the biological activity of deimmunized variants of PE in inhibiting protein synthesis i.e. possess wild type PE biological activity may be performed using the TNT Quick Coupled Transcription Translation Systems assay from PROMEGA Corp. Madison Wis. USA . See PROMEGA Technical Bulletin TB 126 Revised 12 2010 which is incorporated by reference herein.

A preliminary experiment was performed to compare the ability of a PE IL2 fusion protein to inhibit protein synthesis in an in vitro transcription translation assay when a commercially available PE IL fusion protein is translated in vitro following transcription from either a circular plasmid expression vector or a linearized plasmid expression vector. The PE IL2 expression vector in this experiment is referred to as VVN 52431. A few examples of IL2 PE fusion construct are shown in SEQ ID NO 164 165 and 166. The aim of this experiment was to determine if circular or linearized plasmids produced significantly different quantities of PE IL protein in the PROMEGA Corp. TNT Quick Coupled Transcription Translation Systems assay. A commercially available T7 Promoter Luciferase expression vector PROMEGA Corp. hereinafter T7 Luc DNA was used to measure the ability of PE IL2 to inhibit protein synthesis in vitro.

Based on a pilot IVTT experiment it was determined that 0.2 g T7 Luc DNA provided optimal RLU Relative Light Units in a 90 minute IVTT reaction. In this experiment VVN 52431 was linearized using the restriction enzyme Fsp I. Linearized and circular VVN 52431 DNA were used as templates in the IVTT reactions. Reactions were done in triplicate using 0.5 1 and 2 g of DNA. The T7 control reaction was performed using 1 g DNA. Reactions were analyzed via SDS PAGE and by Luciferase assay.

Per manufacturer s instructions Except for the actual transcription translation incubation all handling of the TNT Quick Master Mix was performed at 4 C. Unused Master Mix was refrozen as soon as possible after thawing to minimize loss of translational activity.

Results are shown in . Circular plasmid expression vector encoding PE IL2 fusion protein was slightly better at inhibiting Luciferase protein synthesis compared to linearized plasmid encoding the same at all Luciferase vector VVN 52431 vector ratios . These results also demonstrate the ability of to test and compare the biological activity of PE fusion proteins in inhibiting protein synthesis.

In addition to measuring inhibition of protein synthesis as a measure of light production catalyzed by Luciferase quantitative analysis of inhibition of protein synthesis was also performed by separating polypeptide reaction products on SDS PAGE gels staining and assessing amounts of polypeptide produced data not shown .

Assays such as these may be used to compare the ability of amino acid substituted e.g. deimmunized forms of PE alone or as fusion proteins to retain biological activity such as inhibition of protein synthesis compared to corresponding non amino acid substituted forms of PE alone or as fusion proteins .

Purpose This protocol provides an example of they type of methods which may be used to measure and compare the ribosylation activity i.e. inhibition of protein synthesis of amino acid substituted forms of PE compared to corresponding non amino acid substituted forms of PE.

Background The IVTT assay measures PE mediated inhibition of in vitro transcription translation of a target plasmid T7 Luc. The level of inhibition or lack thereof is determined by chemiluminescent measurement of luciferase activity i.e. the transcribed and translated protein . In this assay a lowered level of transcription and translation and thereby lowered levels of chemiluminescent light output corresponds to increased inhibition of protein synthesis. IVTT can be performed using template DNA encoding PE or by directly using quantified protein. This assay may be used to rank different PE variants against each other and to compare their biological activities to corresponding non amino acid substituted forms.

Materials Test sample Vectors comprising either circular plasmids with an SP6 promoter or linearized plasmids with a T7 promoter.

Per manufacturer s recommendations Except for the actual transcription translation incubation all handling of TNTR Quick Master Mix should be done at 4 C. Any unused Master Mix should be refrozen as soon as possible after thawing to minimize loss of translational activity. Do not freeze thaw the Master Mix more than two times.

Plasmid DNA Dilution Dilute plasmid DNA and Luc plasmid DNA in nuclease free water to final concentration of 0.1 g l.

The immunogenicity of amino acid substituted forms of PE alone or as PE fusion proteins are assessed using methods well known and routinely used by those skilled in the art. For example ELISA assays are used wherein serum is assayed ex vivo following extraction from organisms in which amino acid substituted forms of PE or non amino acid substituted PE alone or as fusion proteins are administered to determine whether or not antibodies that specifically bind the administered protein are produced. It is noted that in the case of PE fusion proteins it is necessary to use as the ELISA assay target antigen not only intact PE fusion proteins i.e. amino acid substituted or non amino acid substituted PE but to also test the PE component and the polypeptide fusion component separately to determine whether or not antibodies produced specifically bind the PE portion or the fusion polypeptide portion e.g. IL2 as used in a previous example of a PE IL2 fusion protein . Accordingly it is most desirable to identify amino acid substituted forms of PE which do not result in host production of antibodies that specifically bind modified forms of PE i.e. deimmunized forms of PE .

Organisms in which amino acid substituted forms of PE may be administered alone or as fusion proteins include for example without limitation mice including transgenic mice expressing human immunoglobulin genes rats rabbits dogs goats sheep horses cows and other bovine species non human primates and humans.

The cytotoxicity of amino acid substituted forms of PE alone or as PE fusion proteins are assessed using methods well known and routinely used by those skilled in the art. For example the cytotoxic effects of amino acid substituted forms of PE alone or as PE fusion proteins administered to cells in vitro or organisms in vivo may be assessed with reference to cytotoxic cell killing effects on target cells organs tissues or tumors against which PE or PE fusion proteins are expected to produce a cytotoxic effect. For example the therapeutically beneficial cytotoxic effects of amino acid substituted PE Mesothelin fusions may be assessed by monitoring and measuring reduction or elimination of tumor or cancer cells or tissues in vitro or in vivo in response to administration of amino acid substituted forms of PE Mesothelin versus wild type PE Mesothelin fusion.

Organisms in which amino acid substituted forms of PE may be administered alone or as fusion proteins include for example without limitation mice including transgenic mice expressing human immunoglobulin genes rats rabbits dogs goats sheep horses cows and other bovine species non human primates and humans.

The Kozak sequence in vector pET14b EMD Millipore catalog 69660 Darmstadt Germany was modified by introducing a linker made up of annealed oligonucleotides 5 CATGGTTCTCCTTCTTAAAGTTAAACAAAATTATTT 3 SEQ ID NO 239 OL2216 in Table 12 and

5 CTAGAAATAATTTTGTTTAACTTTAAGAAGGAGAAC 3 SEQ ID NO 240 OL2217 in Table 12 underlined letters indicate nucleotides changed in the Kozak sequence via NcoI and XbaI restriction sites into vector pET14b resulting in a modified Kozak sequence SEQ ID NO 176 by mutation of three nucleotides at positions 587 to 589. The resulting vector was named pET14b K.

 Kozak sequence gcc gccRccAUGG SEQ ID NO 286 where R is a purine i.e. adenine or guanine three bases upstream of the start codon AUG which is followed by another G . See Kozak 15 20 8125 8148 1987 .

Oligonucleotides for generation of genes encoding amino acid substituted forms of PE are listed as OL2164 to OL2194 and OL2281 to OL2366 in Table 12. A wild type WT PE gene SEQ ID NO 1 was made by gene synthesis and amplified using oligonucleotides 5 ATTGTCCATATGCCAGAAGGCGGTAGCCTGGC 3 SEQ ID NO 215 OL2154 in Table 12 to introduce a NdeI site and

Oligonucleotides for generation of genes encoding amino acid substituted forms Genes encoding amino acid substituted forms of PE were generated using overlapping PCR with the WT PE gene in pET14b K as template. Pairs of primers from the oligonucleotides of Table 12 as noted in the application column of Table 12 were annealed to WT PE DNA and the amino acid substituted genes were PCR amplified using terminal oligonucleotides 5 ATCTCCCTCTAGAAATAATTTTGTTTAACTTTAAGAAG 3 SEQ ID NO 241 OL2268 in Table 12 and

The cell free in vitro transcription translation IVTT assay was performed with a TnT T7 Coupled Reticulocyte Lysate System PROMEGA catalog L4610 following the procedure described in the User s Manual provided in the kit. See PROMEGA Technical Bulletin TB 126 Revised 12 10 pp. 1 28 2010 which is incorporated by reference herein.

WT PE in plasmid pET14b K was used as standard on every plate and tested at concentrations ranging from 0.08 ng to 10 ng in a 12.5 microliter final volume reaction. All test samples were run in triplicate. DNAs encoding WT or amino acid substituted PE in plasmid pET14b K were added to the IVTT reaction mix supplemented with NAD final concentration 0.15 mM Fisher Scientific catalog BPE9746 212 and incubated at 30 C. for 15 min. Following a subsequent cooling step at 4 C. for 5 min 250 ng of T7 Luciferase plasmid Luciferase T7 control DNA supplied in the TnT T7 Coupled Reticulocyte Lysate System was added to each reaction and incubated at 30 C. for 90 min. The reaction was stopped by placing the samples on ice. Samples were analyzed using the STEADY GLO Luciferase Assay PROMEGA catalog E2510 according to the protocol provided by the manufacturer. See PROMEGA Technical Bulletin TM051 revised 9 11 pp. 1 23 2011 which is incorporated by reference herein. Luminescence was measured in a FLUOstar OPTIMA plate reader BMG Labtech Ltd. Aylesbury UK 

A representative result is shown in which shows the results expressed in CPS counts per second as read from the FLUOstar OPTIMA fluorescence plate reader for luciferase activity from IVTT assays of genes encoding either WT PE pIEX02 001 SEQ ID NO 1 or encoding amino acid substituted PE pIEX02 228 SEQ ID NO 177 pIEX02 244 SEQ ID NO 178 pIEX02 246 SEQ ID NO 179 which were expressed as fusion proteins comprising a histidine polymer and a linker sequence preceding a sequence of WT PE or amino acid substituted PE see pIEX02 001 PE WT SEQ ID NO 189 DNA and SEQ ID NO 190 AA pIEX02 228 amino acid substituted PE SEQ ID NO 193 DNA and SEQ ID NO 194 AA pIEX02 244 amino acid substituted PE SEQ ID NO 197 DNA and SEQ ID NO 198 AA pIEX02 246 amino acid substituted PE SEQ ID NO 201 DNA and SEQ ID NO 202 AA . See also Table 13.

For the analysis of various amino acid substituted PEs the potency of each mutated PE in inhibiting IVTT was expressed as relative inhibition exhibited via expression from 2.5 nanograms of DNA encoding amino acid substituted PE compared to expression from 2.5 nanograms WT PE DNA as shown in Table 13 data expressed as inhibition of IVTT for the DNA encoding amino acid substituted PE compared to wild type PE . In order to identify inhibitory amino acid substituted PE polypeptides i.e. genes encoding amino acid substituted forms of PE which inhibit IVTT selected mutations in each T cell epitope as identified in Table 11 were initially tested using various combinations of epitope 5 mutations e.g. corresponding to S241N S241K S241P and S241T in SEQ ID NO 1 along with mutations in either epitope 4 epitopes 4 and 3 epitopes 4 and 1 epitopes 4 and 2 or epitopes 4 and 6 see Table 13 . For all combinations except those including amino acid substitutions in epitope 3 at 1184 SEQ ID NO 1 or 1196 in SEQ ID NO 2 which produced 0 inhibition one or more inhibitory PE polypeptides were identified Table 13 . Note Inhibitory PE polypeptides indicates amino acid substituted forms of PE which exhibit PE biological activity in the inhibition of IVTT . From structural analysis of PE it was noted that residue 1184 SEQ ID NO 1 or 1196 in SEQ ID NO 2 anchor residue 1 Table 11 was located within the active enzymatic site of PE. In view of this result alternative mutations distal to the active site were sought at anchor residues 6 and 9 V189 and R192 in SEQ ID NO 1 or V201 and R204 in SEQ ID NO 2 . Alternative epitope 3 mutations at V189 and R192 in SEQ ID NO 1 were tested in combination with other epitope mutations. These mutations confirmed that inhibitory PE polypeptides with epitope 3 mutations could also be generated Table 13 pIEX02 173 to 248 . A range of combinations of DNAs encoding multiple amino acid substituted forms of PE were tested progressively leading to a final analysis of mutations in four of six five of six and six of six identified immunogenic epitopes. See Table 13 Quadruplicates Quintuplicates and Sextuplicates . In this regard quadruplicate epitope mutations were identified which exhibited IVTT inhibitory activity ranging from 0 to about 70 . Quintuplicate mutations were identified that exhibited IVTT inhibitory activity ranging from about 5 to about 35 . Sextuplicate mutations were identified that exhibited IVTT inhibitory activity ranging from about 5 to about 20 . It is also noted that multiple single double and triple epitope mutations also resulted in amino acid substituted forms of PE exhibiting PE biological activity in the inhibition of IVTT such that the percent inhibitory activity ranged from 0 to 100 or about 100 see Table 13.

Three different candidates i.e. amino acid substituted forms of PE or DNA constructs encoding the same were selected for use as examples in performing subsequent experiments described further herein. In particular additional experiments were performed using the sextuplicate AA substituted candidate pIEX02 244 SEQ ID NO 178 see also Table 13 which retained approximately 20 of the WT PE inhibitory activity. Likewise additional experiments were also performed using the sextuplicate AA substituted candidate pIEX02 246 SEQ ID NO 179 see also Table 13 which retained approximately 8 of the WT PE inhibitory activity and using the quintuplicate AA substituted candidate pIEX02 228 SEQ ID NO 177 see also Table 13 which retained approximately 36 of the WT PE inhibitory activity. These were expressed as fusion proteins comprising a histidine polymer and a linker sequence preceding a sequence of WT PE or amino acid substituted PE see pIEX02 001 PE WT SEQ ID NO 189 DNA and SEQ ID NO 190 AA pIEX02 228 amino acid substituted PE SEQ ID NO 193 DNA and SEQ ID NO 194 AA pIEX02 244 amino acid substituted PE SEQ ID NO 197 DNA and SEQ ID NO 198 AA pIEX02 246 amino acid substituted PE SEQ ID NO 201 DNA and SEQ ID NO 202 AA . These AA substituted PE polypeptides and DNA constructs encoding them may be referenced herein as 228 244 or 246 using simply these three numbers or using these numbers and a prefix or suffix included therewith.

Moreover it is noted that in view of the highly cytotoxic nature of wild type PE IVTT inhibition activity i.e. cytotoxicity as low as about 5 or higher of WT e.g. 8 20 and 36 in amino acid substituted forms of PE may provide a therapeutically effective polypeptide. See for example Thomas et al. Abrogation of Head and Neck Squamous Cell Carcinoma Growth by Epidermal Growth Factor Receptor Ligand Fused to Exotoxin Transforming Growth Factor PE3810 7079 7087 2004 Siegall et al. Cell specific toxicity of a chimeric protein composed of interleukin 6 and exotoxin IL6 PE40 on tumor cells Mol. Cell. Biol. 10 6 2443 2447 1990 and Weldon Pastan A Guide to Taming a Toxin Recombinant Immunotoxins Constructed From Exotoxin A for the Treatment of Cancer FEBS Journal 278 23 4683 4700 2011 .

Ex vivo human T cell assays EPISCREEN see e.g. preceding Examples were performed to assess the immunogenicity of whole proteins corresponding to pIEX02 244 SEQ ID NO 178 pIEX02 246 SEQ ID NO 179 and pIEX02 228 SEQ ID NO 177 Example 10 . In order to avoid direct cytotoxicity to cells used in the assay null mutants were generated for the three candidates and WT PE by overlapping PCR as in Example 9 using primers OL2162 and 2163 Table 12 to introduce an amino acid substitution of E287D in the three candidates and WT PE to give SEQ ID NOs 180 to 183 . Note Null mutants is intended to indicate mutated forms of PE which lack cell cytotoxic biological activity the amino acid substitution used to generate null mutants corresponds to a change of E287D in SEQ ID NO 1 or E299 in SEQ ID NO 2. The E287D SEQ ID NOs 180 to 183 encoding genes were cloned into pET14b K as in Example 9. WT PE sequence is shown in SEQ ID NO 180 pIEX02 228 sequence is shown in SEQ ID NO 181 pIEX02 244 sequence is shown in SEQ ID NO 182 and pIEX02 246 sequence is shown in SEQ ID NO 183. These were expressed as fusion proteins comprising a histidine polymer and a linker sequence preceding a null mutant sequence of WT PE or amino acid substituted PE see pIEX02 001 PE WT null mutant SEQ ID NO 191 DNA and SEQ ID NO 192 AA pIEX02 228 null mutant SEQ ID NO 195 DNA and SEQ ID NO 196 AA pIEX02 244 null mutant SEQ ID NO 199 DNA and SEQ ID NO 200 AA pIEX02 246 null mutant SEQ ID NO 203 DNA and SEQ ID NO 204 AA .

The host cell for expression of the PE E287D genes was an BL21 derivative strain called SHuffle T7 Express NEB catalog C3029H New England Biolabs UK Ltd. Knowl Piece UK which was altered to overexpress the chaperonins GroEL ES by amplification of the GroEL ES operon including its promoter regulatory sites from DH5alpha Invitrogen catalog 18265 017 Life Technologies Ltd. Paisley UK using OL2097 introducing EagI site and OL2098 introducing HindIII site Table 12 . The resulting PCR fragment was subcloned into pACYC184 NEB catalog E4152S which was then transformed into SHuffle T7 with selection for chloramphenicol resistance. The PE E287D SEQ ID NOs 180 to 183 genes in pET14b K were transformed into the SHuffle T7 GroEL ES strain with selection for ampicillin resistance. Single colonies were grown in 2 YT medium Sigma Aldrich catalog Y2627 1KG and protein expression was induced at OD1.0 by adding isopropyl D thio galactoside IPTG to a final concentration of 0.4 mM. Cultures were then grown at 16 degrees C. for 17 h before harvesting by centrifugation. Cell pellets were resuspended in 35 ml of binding buffer 50 mM Tris pH 8.0 500 mM NaCl and 10 mM imidazole supplemented with protease inhibitors cOmplete protease inhibitor tablets Roche catalog 11873580001 Roche Diagnostics Ltd. Burgess Hill UK mixture of several protease inhibitors for inhibition of serine and cysteine proteases . Cells were lysed by sonication SONICATOR Misonix catalog XL2020 Misonix Inc. Farmingdale N.Y. and cell debris and insoluble material removed by centrifugation. Proteins were purified from the soluble fraction by nickel chelate affinity chromatography using HISTRAP FF Crude columns GE Healthcare catalog 11 004 58 GE Healthcare Life Sciences Little Chalfont UK . After loading the columns were washed with 50 mM Tris pH 8.0 containing 500 mM NaCl and 20 mM imidazole and bound protein was eluted with 50 mM Tris pH 8.0 containing 500 mM NaCl and 500 mM imidazole. Following buffer exchange to 20 mM Tris pH 8.0 using Zeba Spin desalting columns 7K MWCO Pierce catalog 89893 a negative purification step was employed using anion exchange chromatography on Q Sepharose 1 ml HISTRAP Q FF column GE Healthcare catalog 17 5053 01 with 20 mM Tris pH 8.0 and an NaCl gradient form 0 M to 1.5 M. For each protein the column flow through was concentrated using an AMICON Ultra centrifugal filter EMD Millipore catalog UFC 800 396 EMD Millipore Feltham UK and further purified by size exclusion chromatography 120 ml HiLoad 16 60 SUPERDEX 75 pg GE Healthcare catalog 28 9893 33 using 1 PBS pH 7.4 PAA catalog H15 002 PAA Laboratories Ltd Yeovil UK . For each protein the protein peak was collected and concentrated.

Endotoxin levels were determined using an ENDOSAFE PTS Portable Test System reader Charles River Laboratories Inc. Wilmington Mass. with ENDOSAFE Licensed PTS Endotoxin cartridges Charles River catalog PTS20F . Endotoxins were reduced to a value below 5 endotoxin units EU mg by repeatedly performing a phase separation using TritonX 114 Aida Y. and Pabst M. J. 132 1990 191 195 . Triton X 114 was removed using PIERCE Detergent Removal Spin Columns according to the manufacturer s provided protocol PIERCE catalog 87779 Thermo Fisher Scientific PIERCE Biotechnology Rockford Ill. see Thermo Scientific Instructions manual 2164.3 . Protein concentration was quantified by absorbance at 280 nm using a BIOMATE 3 UV Visible spectrophotometer Thermo Fisher Scientific and a conversion factor of OD1.0 1.15 mg ml derived from the calculated molar extinction coefficient of 6 His PE Pace C. N. et al. Protein Science 1995 4 2411 2423 .

Ex vivo human T cell assays EPISCREEN were performed using PBMC isolated from healthy community donor buffy coats as in Example 2. A cohort of 20 donors was selected to best represent the number and frequency of HLA DR allotypes expressed in the world population. The haplotypes of the 20 donors in the assay is shown in Table 14. PBMCs from each donor were thawed counted and viability assessed. Cells were revived in room temperature AIM V culture medium INVITROGEN Paisley UK washed and resuspended in AIM V to 4 6 10PBMC ml. For each donor 1 ml of cells were dispensed into multiple wells of a 24 well plate. 0.5 ml of proteins were added at 50 micrograms ml per sample together with 0.5 ml of AIM V culture medium. For each donor a reproducibility control cells incubated with 100 micrograms ml keyhole limpet hemocyanin KLH an intermediate positive control expected to give 20 30 T cell responses of humanized A33 antibody Welt et al. Clinical Cancer Research 9 2003 p1338 1343 cells were incubated with 50 micrograms ml humanized A33 and a culture medium only control well were also included. Cultures were incubated for a total of 8 days at 37 C. with 5 CO.

For the T cell proliferation assay on days 5 6 7 and 8 the cells in each well were gently resuspended and triplicate 100 microliter aliquots were transferred to each well of a round bottomed 96 well plate. The cultures were pulsed with 0.75 microCi 3H Thymidine PERKIN ELMER Beaconsfield UK in 100 microliters AIM V culture medium and incubated for a further 18 hours before harvesting onto filter mats Perkin Elmer using a TOMTEC HARVESTER 96 Mach III cell harvester TOMTEC Inc. Hamden Conn. USA . Counts per minute cpm for each well were determined using MELTILEX solid scintillator PERKIN ELMER Life and Analytical Sciences Shelton Conn. USA via scintillation counting on a Wallac 1450 Microbeta Trilux Microplate Scintillation and Luminescence Counter Perkin Elmer in paralux low background counting.

For proliferation assays an empirical Stimulation Index S threshold of equal to or greater than 2 SI 2.0 was used whereby samples inducing proliferative responses above this threshold at any day after addition of protein were deemed positive. The Stimulation Index is a ratio of stimulated proliferative response compared to a background index an SI of 1 background or noise . For the triplicate proliferation data for each time point with each protein the significance p

The results of the proliferation assay are shown in Table 15. The results demonstrate a significantly reduced level of T cell responses from the amino acid substituted PE molecules pIEX02 228 SEQ ID NO 181 5 donor responses pIEX02 244 SEQ ID NO 182 10 donor responses and pIEX02 246 SEQ ID NO 183 20 donor responses compared to WT PE SEQ ID NO 180 which induced T cell responses in 70 of donors.

In addition to the proliferation assay additional analysis of the cytokines IL 2 and IL 6 was performed using aliquots of culture supernatant taken on day 6. The analysis was performed using the BD Cytometric Bead Array CBA Enhanced Sensitivity Flex Set Systems for IL 2 and IL 6 BD Bioscience Oxford UK according to the manufacturer s instructions. The enhanced sensitivity standards from the CBA kit were reconstituted and serially diluted before adding 50 microliters of supernatant or standard to 20 microliters of mixed capture beads in 96 well filter plates Millipore Watford UK and incubating for 2 hours. Mixed human detection reagent 20 microliters was then added to each well and incubated for a further 2 hours. Plates were washed twice and enhanced detection 100 microliters reagent added to each well for a final 1 hour incubation. Plates were washed before reading on an Accuri C6 instrument BD Biosciences .

Data was analysed using FCAP v3.0 software BD Biosciences . For each individual donor data was expressed as pg ml of cytokine for each donor and plotted on a log scale with a median of cytokine levels depicted as a line. The results are shown in which shows a significantly reduced level of the cytokines IL 2 and IL 6 from the amino acid substituted PE molecules pIEX02 228 SEQ ID NO 181 pIEX02 244 SEQ ID NO 182 and pIEX02 246 SEQ ID NO 183 compared to WT PE SEQ ID NO 180 .

The proliferation and cytokine results both independently demonstrate that the amino acid substitutions in PE result in greatly reduced level of T cell responses when using amino acid substituted forms of PE. These results considered and expected to correlate with low or reduced PE immunogenicity in human subjects.

Amino acid substituted forms of PE and WT PE may be generated as in Example 11. For a dendritic cell cytotoxicity assay PBMC are isolated from healthy community donor buffy coats preferably from blood drawn within 24 hours for example by Lymphoprep Axis shield Dundee UK density centrifugation. To prepare monocyte derived dendritic cells DC CD14 cells monocytes may be isolated from donor PBMC preparations using Miltenyi Monocyte Isolation Kit II human and LS columns Miltenyi Biotech GmbH Bergisch Gladbach Germany catalog 130 091 153 . Monocytes are resuspended in an appropriate culture medium such as AIM V cell culture medium supplemented with 1000 IU ml IL 4 and 1000 IU ml GM CSF DC culture medium to 4 6 10cells ml and then distributed in 24 well plates e.g. 2 ml final culture volume . Cells are fed on day 2 by half volume DC culture medium change. On day 3 amino acid substituted PE and WT PE proteins are added to semi mature DC to selected final concentrations such as 1 micrograms ml or 10 micrograms ml. Semi mature DC are incubated for a period of time such as 24 72 hours after which cells are assessed for cytotoxicity by viability such as via use of Trypan Blue Sigma Dorset UK dye exclusion and by propidium iodide PI and Annexin V staining Invitrogen Paisley UK followed by FACS analysis.

WT PE and amino acid substituted PE encoded by pIEX02 244 SEQ ID NO 178 are fused to an anti CD22 single chain Fv scFv . The VH and VL V regions of RFB4 Campana D. et al. 134 1524 1530 1985 Mansfield E. et al. Blood 90 2020 2026 1997 are synthesized. RFB4 VH is amplified using oligonucleotides OL2043 and OL2044 Table 12 . RFB4 and VL Vk is amplified using oligonucleotides OL2045 and OL2046 Table 12 . The RFB4 scFv is obtained using a pull through PCR reaction using oligonucleotides OL2047 introducing a NcoI site and OL2048 introducing a XhoI site. The resultant PCR product is subcloned into pET14b K via NcoI and XhoI restriction enzymes Fermentas catalog FD0573 and FD0695 respectively .

The gene encoding RFB4 scFv is fused to genes encoding either WT PE or amino acid substituted PE encoded by pIEX02 244 SEQ ID NO 178 having a C terminal 8 His tag followed by the sorting signal EDLK to give fusion sequences SEQ ID NO 184 and SEQ ID NO 186 respectively which are cloned into the expression vector pET14b K by fusion PCR. To create these RFB4 PE fusions a fusion PCR is carried out. The RFB4 scFv gene is amplified from pET14b RFB4 using oligonucleotides OL2318 introducing a NcoI site and OL2320 Table 12 . The WT PE or the lead amino acid substituted PE genes are amplified from pET14b K WT PE or pET14b K 244 PE respectively using oligonucleotides OL2321 and OL2322 introducing a N terminal 8 His EDLK and a XhoI site Table 12 . Both scFv and PE genes are fused by performing a PCR with oligonucleotides OL2318 and OL2322 Table 12 . The resulting full length fragments are subcloned into pET14b K using NcoI and XhoI restriction enzymes. Plasmids are transformed into BL21 DE3 EMD Millipore Feltham UK and clones are inoculated into 2TY Amp and grown overnight at 37 C. Two ml of overnight culture is inoculated into 350 ml 2TY Amp media in a 1 L flask and grown to OD 1 before addition of IPTG Sigma to 1 mM final concentration . Cultures are grown overnight at 30 degrees C. overnight and centrifuged at 10000 rpm. Bacterial pellets are frozen at 80 until ready to use.

Pellets are defrosted on ice extracted with 10 ml B PER Bacterial Protein Extraction Reagent PIERCE Biotechnology Rockford Ill. Thermo Scientific Hemel Hempstead UK containing Lysozyme and DNasel both Thermo Scientific and rotated for 1 hour at room temperature. Samples are then centrifuged at 10000 rpm and supernatants are discarded. Each pellet is resuspended in 5 ml B PER containing Lysozyme and DNasel as above and extracted for an additional 30 min at room temperature. After centrifugation pellets are pooled and washed successively with Wash Buffer A 50 mM Tris HCl pH 8.0 100 mM NaCl 1 mM EDTA 0.5 M urea and 1.0 Triton X 100 Wash Buffer B Buffer A but without urea and twice with Wash Buffer C Buffer A but without urea or Triton X 100 . After final wash insoluble pellets are stored at 80 C. cOmplete mini EDTA protease inhibitors Roche Diagnostics Ltd. are included at each step.

Pellets are resuspended in 10 ml Solubilisation Buffer 50 mM Tris HCl pH 8.0 100 mM NaCl 8 M Urea and 1 mM DTT . ODis determined and the samples are diluted to approximately 1 mg ml in Solubilisation Buffer. Protein samples are allowed to denature for 4 hours at room temperature and centrifuged at 10000 rpm to remove insoluble debris. 10 ml of each solubilised protein samples is injected into a pre soaked 12 ml 3K MWCO cut off SLIDE A LYSER Dialysis Cassette dialysis device Thermo Scientific PIERCE Biotechnology Inc. IL USA and dialyzed by placement overnight in a beaker containing 2.5 L Refolding Buffer A 50 mM Tris HCl pH 8.0 100 mM NaCl 5 mM reduced glutathione 1 mM oxidised glutathione 0.1 M arginine 4 M urea . Dialysis buffer is replaced with in order 2.5 L Refolding Buffer B Buffer A with 2 M urea 2.5 L Refolding Buffer C Buffer A with 1 M urea and 5 L Refolding Buffer D Buffer A without urea or arginine for a minimum of 4 hours at each step.

Each sample is recovered from the dialysis cassette buffer exchanged into 50 mM 2 N morpholino ethanesulfonic acid MES pH 6.2 using PD10 desalting columns GE Healthcare Little Chalfont UK and loaded onto a 1 ml SP FF Anion Exchange column GE Healthcare . Each column is washed with 50 mM MES pH 6.2 before eluting using a linear 0M to 1M NaCl gradient in 50 mM MES pH6.2. Protein containing fractions are pooled and run through a pre equilibrated 16 60 Size Exclusion column GE Healthcare using 1 PBS as running buffer. Fractions containing the main protein peaks are collected pooled and concentrated to approximately 1 ml filter sterilized and quantified.

For cytotoxicity analysis Raji cells ATCC CCL 86 are propagated in growth medium RPMI 1640 10 FBS 1 Pen Strep and harvested during mid log growth phase. Cells are diluted to 1 10cells ml in growth medium and 50 microliter aliquots are dispensed per well in white walled clear bottom 96 well plates CORNING catalogue 3610 FISHER SCIENTIFIC Loughborough UK . Each protein concentration 8 4 fold dilutions from 500 nanograms ml is tested in triplicate wells and controls containing cells or growth medium only are also included. Test protein is diluted to 2 desired concentration in growth medium. 50 microliters of the test protein dilutions or controls are added to the Raji cells and plates are incubated 72 hrs in a humidified cell culture incubator 37 C. 5 CO . After incubation plates are equilibrated at room temperature for 10 min. CELLTITER GLO PROMEGA catalogue G7571 is prepared according to manufacturer s instructions and 100 microliters is added per well. Plates are incubated for 10 min before reading via a FLUOstar OPTIMA fluorescence plate reader BMG Labtech Ltd. Aylesbury UK also known as a fluorometer .

